{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "f104c585-ecad-41ff-abea-fa7333686576",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3.6.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Running on Apache Spark version 2.4.1\n",
      "SparkUI available at http://vega.local:4040\n",
      "Welcome to\n",
      "     __  __     <>__\n",
      "    / /_/ /__  __/ /\n",
      "   / __  / _ `/ / /\n",
      "  /_/ /_/\\_,_/_/_/   version 0.2.64-1ef70187dc78\n",
      "LOGGING: writing to /enadisk/maison/sage2016/weber/PycharmProjects/MISTIC_CAGI6/hail-20210928-1954-0.2.64-1ef70187dc78.log\n"
     ]
    }
   ],
   "source": [
    "from platform import python_version\n",
    "print(python_version())\n",
    "import numpy as np\n",
    "import subprocess\n",
    "\n",
    "import pandas as pd\n",
    "import dask.dataframe as dd\n",
    "import hail as hl\n",
    "hl.init(min_block_size=256)\n",
    "\n",
    "import os\n",
    "os.environ['PYSPARK_SUBMIT_ARGS'] = \"--driver-memory 700G pyspark-shell\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "00221fe0-9ba2-418c-9587-894726e3483a",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Remap assembly\n",
    "\n",
    "def remap_subprocess_fct(input_bed, output_bed, perl_location=\"/usr/local/bin/perl\", remap_location=\"/biolo/ngs/remap/remap_api.pl\"):\n",
    "\n",
    "    subprocess_args = [\n",
    "        perl_location,\n",
    "        remap_location,\n",
    "        \"--mode\",\n",
    "        \"asm-asm\",\n",
    "        \"--from\",\n",
    "        \"GCF_000001405.25\",\n",
    "        \"--dest\",\n",
    "        \"GCF_000001405.26\",\n",
    "        \"--annotation\",\n",
    "        input_bed,\n",
    "        \"--annot_out\",\n",
    "        output_bed,\n",
    "        \"--report_out\",\n",
    "        \"report\",\n",
    "        \"--in_format\",\n",
    "        \"bed\",\n",
    "        \"--out_format\",\n",
    "        \"bed\",\n",
    "    ]\n",
    "    p1 = subprocess.Popen(subprocess_args)\n",
    "    p1.wait()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "id": "332e8b4d-3ecf-4338-a739-0fcf42b19874",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/weber/.conda/envs/hail/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3072: DtypeWarning: Columns (0) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n",
      "100%|██████████| 24/24 [01:22<00:00,  3.44s/it]\n"
     ]
    }
   ],
   "source": [
    "# CCRS IMPORT & REFORMAT\n",
    "\n",
    "ccr = pd.read_csv('/gstock/biolo_datasets/variation/benchmark/Databases/CCR/ccrs_complete.bedGraph.gz', compression='gzip', sep='\\t', names=['CHR', 'START', 'END', 'CCR'])\n",
    "from tqdm import tqdm\n",
    "for chromosome in tqdm(ccr.CHR.unique()):\n",
    "    ccr.loc[ccr['CHR'] == chromosome].to_csv('/gstock/biolo_datasets/variation/benchmark/Databases/CCR/CCR_split/ccr.chr{}.bedGraph.gz'.format(str(chromosome)), compression='gzip', sep='\\t', header=False, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 176,
   "id": "8e14db55-862e-43b3-a3cc-7658b2106860",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# CCR REFORMAT\n",
    "\n",
    "ccrs_table_pd = pd.read_csv(\"/gstock/biolo_datasets/variation/benchmark/Databases/CCR/CCR_split/38/ccr.complete.bedGraph.gz\", compression='gzip', sep='\\t', names=['CHR', 'START', 'END', 'CCR'])\n",
    "chroms = ['{}'.format(i) for i in list(range(1,23)) + ['X']]\n",
    "ccrs_table_pd.CHR = ccrs_table_pd.CHR.astype(str)\n",
    "ccrs_table_pd.loc[ccrs_table_pd['CHR'].isin(chroms), 'CHR'] = 'chr' + ccrs_table_pd.loc[ccrs_table_pd['CHR'].isin(chroms), 'CHR']\n",
    "ccrs_table_pd.loc[ccrs_table_pd['CHR'].str.contains('chr')].to_csv('/gstock/biolo_datasets/variation/benchmark/Databases/CCR/CCR_split/38/ccr.complete.38.bedGraph.gz',compression='gzip', sep='\\t', header=False, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "id": "13c3393f-efc6-4e35-9bb1-6d579b8c6a0e",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2021-09-25 23:50:07 Hail: INFO: Reading table without type imputation\n",
      "  Loading field 'f0' as type str (user-supplied)\n",
      "  Loading field 'f1' as type int32 (user-supplied)\n",
      "  Loading field 'f2' as type int32 (user-supplied)\n",
      "  Loading field 'f3' as type str (user-supplied)\n"
     ]
    }
   ],
   "source": [
    "# CCR IMPORT IN HAIL\n",
    "\n",
    "chroms = {'{}'.format(i) : 'chr{}'.format(i) for i in list(range(1,23)) + ['X']}\n",
    "ccrs_table = hl.import_bed(\n",
    "    \"/gstock/biolo_datasets/variation/benchmark/Databases/CCR/CCR_split/38/ccr.complete.38.bedGraph.gz\",\n",
    "    reference_genome=\"GRCh38\",\n",
    "    force_bgz=True,\n",
    "    contig_recoding=chroms,\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "id": "a803276b-3452-4787-843a-4cdc45f0c9b1",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2021-09-25 23:49:44 Hail: INFO: Reading table without type imputation\n",
      "  Loading field '#CHROM (GRCh37 assembly)' as type str (not specified)\n",
      "  Loading field 'POS' as type str (not specified)\n",
      "  Loading field 'REF' as type str (not specified)\n",
      "  Loading field 'ALT' as type str (not specified)\n",
      "  Loading field 'MISTIC_score' as type str (not specified)\n",
      "  Loading field 'MISTIC_pred' as type str (not specified)\n"
     ]
    }
   ],
   "source": [
    "\n",
    "# MISTIC EXISTING SCORES IMPORT IN HAIL\n",
    "\n",
    "mistic_hail = hl.import_table('/gstock/biolo_datasets/variation/benchmark/MISTIC/PRECOMPUTING_CADD/2021/MISTIC_GRCh37_website_2021.tsv.bgz', force_bgz=True)\n",
    "\n",
    "mistic_hail = mistic_hail.annotate(\n",
    "    ID  = mistic_hail['#CHROM (GRCh37 assembly)'].replace('chr', '') + '-' + mistic_hail['POS'] + '-' + mistic_hail['REF'] + '-' + mistic_hail['ALT']\n",
    ")\n",
    "# mistic_hail = mistic_hail.filter(mistic_hail['#CHROM (GRCh37 assembly)'] == '10')\n",
    "mistic_hail = mistic_hail.key_by('ID')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "id": "ce84143f-4873-4b87-9e27-199b9923828f",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# dbNSFP IMPORT IN HAIL\n",
    "\n",
    "chroms = {'{}'.format(i) : 'chr{}'.format(i) for i in list(range(1,23)) + ['X', 'Y', 'M']}\n",
    "# chroms = {'{}'.format(i) : 'chr{}'.format(i) for i in [10]}\n",
    "dbnsfp_vcf_hail = hl.import_vcf('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.vcf.gz', reference_genome=\"GRCh38\", force_bgz=True, contig_recoding=chroms)\n",
    "# dbnsfp_test['id'] = dbnsfp_test[\"#chr\"].astype(str) + '-' + dbnsfp_test[\"pos(1-coor)\"].astype(str) + '-' + dbnsfp_test[\"ref\"].astype(str) + '-' + dbnsfp_test[\"alt\"].astype(str) + '-' + dbnsfp_test[\"hg19_chr\"].astype(str) + '-' + dbnsfp_test[\"hg19_pos(1-based)\"].astype(str) \n",
    "\n",
    "dbnsfp_vcf_hail = dbnsfp_vcf_hail.annotate_rows(\n",
    "    CSQ_split = dbnsfp_vcf_hail.info.CSQ.map(lambda x: x.split('\\|'))\n",
    ")\n",
    "\n",
    "dbnsfp_vcf_hail = dbnsfp_vcf_hail.annotate_rows(\n",
    "    Allele = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[0]),\n",
    "    Consequence = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[1]),\n",
    "    IMPACT = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[2]),\n",
    "    SYMBOL = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[3]),\n",
    "    Gene = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[4]),\n",
    "    CONDEL = dbnsfp_vcf_hail.CSQ_split.map(lambda x: x[36]),\n",
    ")\n",
    "\n",
    "dbnsfp_vcf_hail = dbnsfp_vcf_hail.annotate_rows(\n",
    "    MISSENSE = dbnsfp_vcf_hail.Consequence.contains('missense_variant'),\n",
    "    STOPGAINED = dbnsfp_vcf_hail.Consequence.contains('stop_gained'),\n",
    "    STOPLOST = dbnsfp_vcf_hail.Consequence.contains('stop_lost'),\n",
    "    STARTLOST = dbnsfp_vcf_hail.Consequence.contains('start_lost'),\n",
    "#     INFO = dbnsfp_vcf_hail.info.INFO[0],\n",
    "    ID_raw = dbnsfp_vcf_hail.info.INFO[0].split('-'),\n",
    ")\n",
    "\n",
    "dbnsfp_vcf_hail = dbnsfp_vcf_hail.annotate_rows(\n",
    "    ID = dbnsfp_vcf_hail.ID_raw[4] + '-' + dbnsfp_vcf_hail.ID_raw[5] + '-' + dbnsfp_vcf_hail.ID_raw[2] + '-' + dbnsfp_vcf_hail.ID_raw[3]\n",
    "    \n",
    ")\n",
    "\n",
    "dbnsfp_vcf_hail = dbnsfp_vcf_hail.annotate_rows(\n",
    "    MISTIC = hl.float(mistic_hail[dbnsfp_vcf_hail.ID].MISTIC_score),\n",
    "    CCR = hl.float(ccrs_table[dbnsfp_vcf_hail.locus].target),\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5e96264a-e504-499b-b564-2bf461827bea",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2021-09-25 23:52:30 Hail: INFO: Ordering unsorted dataset with network shuffle\n",
      "2021-09-25 23:53:05 Hail: INFO: Ordering unsorted dataset with network shuffle\n",
      "2021-09-25 23:55:28 Hail: INFO: Ordering unsorted dataset with network shuffle\n",
      "2021-09-25 23:56:20 Hail: INFO: Ordering unsorted dataset with network shuffle\n"
     ]
    }
   ],
   "source": [
    "# CONVERT WITH HAIL VCF TO CSV \n",
    "\n",
    "dbnsfp_vcf_hail.rows().export('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.csv.bgz')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "01c934a7-bb06-4365-8d5b-2dcd53399638",
   "metadata": {},
   "outputs": [],
   "source": [
    "# IMPORT INTO PANDAS\n",
    "\n",
    "dbnsfp_vcf_hail_pandas = pd.read_csv('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.csv.bgz', compression='gzip', sep='\\t')\n",
    "dbnsfp_vcf_hail_pandas"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "637e869a-27ab-4b49-966e-aaa5fdcc29a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# CONVERT TO PARQUET FOR MORE EFFICIENT R/W\n",
    "\n",
    "dbnsfp_vcf_hail_pandas.to_parquet('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.parquet', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4d4edf5e",
   "metadata": {},
   "source": [
    "# dbNSFP PANDAS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "5fe0cdca-1e7e-448b-a8ca-3eb7b319cd50",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>locus</th>\n",
       "      <th>alleles</th>\n",
       "      <th>rsid</th>\n",
       "      <th>qual</th>\n",
       "      <th>filters</th>\n",
       "      <th>info</th>\n",
       "      <th>CSQ_split</th>\n",
       "      <th>Allele</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>...</th>\n",
       "      <th>Gene</th>\n",
       "      <th>CONDEL</th>\n",
       "      <th>MISSENSE</th>\n",
       "      <th>STOPGAINED</th>\n",
       "      <th>STOPLOST</th>\n",
       "      <th>STARTLOST</th>\n",
       "      <th>ID_raw</th>\n",
       "      <th>ID</th>\n",
       "      <th>MISTIC</th>\n",
       "      <th>CCR</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"ENSG00000240361\",\"ENSG00000186092\"]</td>\n",
       "      <td>[\"\",\"neutral(0.000)\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"C\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"ENSG00000240361\",\"ENSG00000186092\"]</td>\n",
       "      <td>[\"\",\"neutral(0.349)\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"G\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"ENSG00000240361\",\"ENSG00000186092\"]</td>\n",
       "      <td>[\"\",\"neutral(0.000)\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"T\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1:65566</td>\n",
       "      <td>[\"T\",\"A\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"A\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"A\",\"A\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"ENSG00000240361\",\"ENSG00000186092\"]</td>\n",
       "      <td>[\"\",\"neutral(0.445)\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65566\",\"T\",\"A\",\"1\",\"65566\"]</td>\n",
       "      <td>1-65566-T-A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1:65566</td>\n",
       "      <td>[\"T\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"ENSG00000186092\",\"ENSG00000240361\"]</td>\n",
       "      <td>[\"neutral(0.406)\",\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65566\",\"T\",\"C\",\"1\",\"65566\"]</td>\n",
       "      <td>1-65566-T-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81782918</th>\n",
       "      <td>chrM:14672</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...</td>\n",
       "      <td>[\"G\"]</td>\n",
       "      <td>[\"intergenic_variant\"]</td>\n",
       "      <td>[\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"M\",\"14672\",\"A\",\"G\",\"M\",\"14673\"]</td>\n",
       "      <td>M-14673-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81782919</th>\n",
       "      <td>chrM:14672</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...</td>\n",
       "      <td>[\"T\"]</td>\n",
       "      <td>[\"intergenic_variant\"]</td>\n",
       "      <td>[\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"M\",\"14672\",\"A\",\"T\",\"M\",\"14673\"]</td>\n",
       "      <td>M-14673-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81782920</th>\n",
       "      <td>chrM:14673</td>\n",
       "      <td>[\"T\",\"A\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"A\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...</td>\n",
       "      <td>[\"A\"]</td>\n",
       "      <td>[\"intergenic_variant\"]</td>\n",
       "      <td>[\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"M\",\"14673\",\"T\",\"A\",\"M\",\"14674\"]</td>\n",
       "      <td>M-14674-T-A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81782921</th>\n",
       "      <td>chrM:14673</td>\n",
       "      <td>[\"T\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...</td>\n",
       "      <td>[\"C\"]</td>\n",
       "      <td>[\"intergenic_variant\"]</td>\n",
       "      <td>[\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"M\",\"14673\",\"T\",\"C\",\"M\",\"14674\"]</td>\n",
       "      <td>M-14674-T-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81782922</th>\n",
       "      <td>chrM:14673</td>\n",
       "      <td>[\"T\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...</td>\n",
       "      <td>[\"G\"]</td>\n",
       "      <td>[\"intergenic_variant\"]</td>\n",
       "      <td>[\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>[\"\"]</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"M\",\"14673\",\"T\",\"G\",\"M\",\"14674\"]</td>\n",
       "      <td>M-14674-T-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>81782923 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               locus    alleles  rsid  qual  filters  \\\n",
       "0         chr1:65565  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "1         chr1:65565  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "2         chr1:65565  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "3         chr1:65566  [\"T\",\"A\"]   NaN -10.0      NaN   \n",
       "4         chr1:65566  [\"T\",\"C\"]   NaN -10.0      NaN   \n",
       "...              ...        ...   ...   ...      ...   \n",
       "81782918  chrM:14672  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "81782919  chrM:14672  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "81782920  chrM:14673  [\"T\",\"A\"]   NaN -10.0      NaN   \n",
       "81782921  chrM:14673  [\"T\",\"C\"]   NaN -10.0      NaN   \n",
       "81782922  chrM:14673  [\"T\",\"G\"]   NaN -10.0      NaN   \n",
       "\n",
       "                                                       info  \\\n",
       "0         {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "1         {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "2         {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "3         {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "4         {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "...                                                     ...   \n",
       "81782918  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "81782919  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "81782920  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "81782921  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "81782922  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "\n",
       "                                                  CSQ_split     Allele  \\\n",
       "0         [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "1         [[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"G\",\"G\"]   \n",
       "2         [[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"T\",\"T\"]   \n",
       "3         [[\"A\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"A\",\"A\"]   \n",
       "4         [[\"C\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...  [\"C\",\"C\"]   \n",
       "...                                                     ...        ...   \n",
       "81782918  [[\"G\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...      [\"G\"]   \n",
       "81782919  [[\"T\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...      [\"T\"]   \n",
       "81782920  [[\"A\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...      [\"A\"]   \n",
       "81782921  [[\"C\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...      [\"C\"]   \n",
       "81782922  [[\"G\",\"intergenic_variant\",\"MODIFIER\",\"\",\"\",\"\"...      [\"G\"]   \n",
       "\n",
       "                                       Consequence               IMPACT  ...  \\\n",
       "0         [\"downstream_gene_variant\",\"start_lost\"]  [\"MODIFIER\",\"HIGH\"]  ...   \n",
       "1         [\"downstream_gene_variant\",\"start_lost\"]  [\"MODIFIER\",\"HIGH\"]  ...   \n",
       "2         [\"downstream_gene_variant\",\"start_lost\"]  [\"MODIFIER\",\"HIGH\"]  ...   \n",
       "3         [\"downstream_gene_variant\",\"start_lost\"]  [\"MODIFIER\",\"HIGH\"]  ...   \n",
       "4         [\"start_lost\",\"downstream_gene_variant\"]  [\"HIGH\",\"MODIFIER\"]  ...   \n",
       "...                                            ...                  ...  ...   \n",
       "81782918                    [\"intergenic_variant\"]         [\"MODIFIER\"]  ...   \n",
       "81782919                    [\"intergenic_variant\"]         [\"MODIFIER\"]  ...   \n",
       "81782920                    [\"intergenic_variant\"]         [\"MODIFIER\"]  ...   \n",
       "81782921                    [\"intergenic_variant\"]         [\"MODIFIER\"]  ...   \n",
       "81782922                    [\"intergenic_variant\"]         [\"MODIFIER\"]  ...   \n",
       "\n",
       "                                           Gene                 CONDEL  \\\n",
       "0         [\"ENSG00000240361\",\"ENSG00000186092\"]  [\"\",\"neutral(0.000)\"]   \n",
       "1         [\"ENSG00000240361\",\"ENSG00000186092\"]  [\"\",\"neutral(0.349)\"]   \n",
       "2         [\"ENSG00000240361\",\"ENSG00000186092\"]  [\"\",\"neutral(0.000)\"]   \n",
       "3         [\"ENSG00000240361\",\"ENSG00000186092\"]  [\"\",\"neutral(0.445)\"]   \n",
       "4         [\"ENSG00000186092\",\"ENSG00000240361\"]  [\"neutral(0.406)\",\"\"]   \n",
       "...                                         ...                    ...   \n",
       "81782918                                   [\"\"]                   [\"\"]   \n",
       "81782919                                   [\"\"]                   [\"\"]   \n",
       "81782920                                   [\"\"]                   [\"\"]   \n",
       "81782921                                   [\"\"]                   [\"\"]   \n",
       "81782922                                   [\"\"]                   [\"\"]   \n",
       "\n",
       "         MISSENSE  STOPGAINED  STOPLOST  STARTLOST  \\\n",
       "0           False       False     False       True   \n",
       "1           False       False     False       True   \n",
       "2           False       False     False       True   \n",
       "3           False       False     False       True   \n",
       "4           False       False     False       True   \n",
       "...           ...         ...       ...        ...   \n",
       "81782918    False       False     False      False   \n",
       "81782919    False       False     False      False   \n",
       "81782920    False       False     False      False   \n",
       "81782921    False       False     False      False   \n",
       "81782922    False       False     False      False   \n",
       "\n",
       "                                     ID_raw           ID MISTIC  CCR  \n",
       "0         [\"1\",\"65565\",\"A\",\"C\",\"1\",\"65565\"]  1-65565-A-C    NaN  NaN  \n",
       "1         [\"1\",\"65565\",\"A\",\"G\",\"1\",\"65565\"]  1-65565-A-G    NaN  NaN  \n",
       "2         [\"1\",\"65565\",\"A\",\"T\",\"1\",\"65565\"]  1-65565-A-T    NaN  NaN  \n",
       "3         [\"1\",\"65566\",\"T\",\"A\",\"1\",\"65566\"]  1-65566-T-A    NaN  NaN  \n",
       "4         [\"1\",\"65566\",\"T\",\"C\",\"1\",\"65566\"]  1-65566-T-C    NaN  NaN  \n",
       "...                                     ...          ...    ...  ...  \n",
       "81782918  [\"M\",\"14672\",\"A\",\"G\",\"M\",\"14673\"]  M-14673-A-G    NaN  NaN  \n",
       "81782919  [\"M\",\"14672\",\"A\",\"T\",\"M\",\"14673\"]  M-14673-A-T    NaN  NaN  \n",
       "81782920  [\"M\",\"14673\",\"T\",\"A\",\"M\",\"14674\"]  M-14674-T-A    NaN  NaN  \n",
       "81782921  [\"M\",\"14673\",\"T\",\"C\",\"M\",\"14674\"]  M-14674-T-C    NaN  NaN  \n",
       "81782922  [\"M\",\"14673\",\"T\",\"G\",\"M\",\"14674\"]  M-14674-T-G    NaN  NaN  \n",
       "\n",
       "[81782923 rows x 21 columns]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dbnsfp_vcf_hail_pandas = pd.read_parquet('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.parquet')\n",
    "dbnsfp_vcf_hail_pandas"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a4251abc-d022-4978-9f35-52628c994eca",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# REFORMAT dbNSFP with pandas\n",
    "\n",
    "# dbnsfp_vcf_hail_dask = dd.read_parquet('/gstock/MISTIC/CAGI6/dbnsfp_complete.vep.parquet')\n",
    "dbnsfp_vcf_hail_pandas['CHROM'] = dbnsfp_vcf_hail_pandas['locus'].apply(lambda r: r.split(':')[0].replace('chr', ''),)\n",
    "dbnsfp_vcf_hail_pandas['CHROM_37'] = dbnsfp_vcf_hail_pandas['ID'].apply(lambda r: r.split('-')[0], )\n",
    "dbnsfp_vcf_hail_pandas['FULL_ID'] = dbnsfp_vcf_hail_pandas['ID_raw'].apply(lambda r: \"-\".join(eval(r)), )\n",
    "\n",
    "# Select missense\n",
    "dbnsfp_vcf_hail_pandas['Missense_index'] = dbnsfp_vcf_hail_pandas['Consequence'].apply(lambda r: [j for j,e in enumerate(eval(r)) if 'missense' in e])\n",
    "dbnsfp_vcf_hail_pandas.loc[dbnsfp_vcf_hail_pandas['Consequence'].str.contains('missense'), 'MISSENSE'] = True\n",
    "# dbnsfp_vcf_hail_dask"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "289e04e3-a5d8-4df0-b63d-2cc7336744f9",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>locus</th>\n",
       "      <th>alleles</th>\n",
       "      <th>rsid</th>\n",
       "      <th>qual</th>\n",
       "      <th>filters</th>\n",
       "      <th>info</th>\n",
       "      <th>CSQ_split</th>\n",
       "      <th>Allele</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>...</th>\n",
       "      <th>STOPLOST</th>\n",
       "      <th>STARTLOST</th>\n",
       "      <th>ID_raw</th>\n",
       "      <th>ID</th>\n",
       "      <th>MISTIC</th>\n",
       "      <th>CCR</th>\n",
       "      <th>CHROM</th>\n",
       "      <th>CHROM_37</th>\n",
       "      <th>FULL_ID</th>\n",
       "      <th>Missense_index</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"C\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65565-A-C-1-65565</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"G\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65565-A-G-1-65565</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1:65565</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65565\",\"A\",\"T\",\"1\",\"65565\"]</td>\n",
       "      <td>1-65565-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65565-A-T-1-65565</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1:65566</td>\n",
       "      <td>[\"T\",\"A\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"A\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"A\",\"A\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65566\",\"T\",\"A\",\"1\",\"65566\"]</td>\n",
       "      <td>1-65566-T-A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65566-T-A-1-65566</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1:65566</td>\n",
       "      <td>[\"T\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65566\",\"T\",\"C\",\"1\",\"65566\"]</td>\n",
       "      <td>1-65566-T-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65566-T-C-1-65566</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>chr1:65566</td>\n",
       "      <td>[\"T\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65566\",\"T\",\"G\",\"1\",\"65566\"]</td>\n",
       "      <td>1-65566-T-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65566-T-G-1-65566</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>chr1:65567</td>\n",
       "      <td>[\"G\",\"A\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"A\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...</td>\n",
       "      <td>[\"A\",\"A\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65567\",\"G\",\"A\",\"1\",\"65567\"]</td>\n",
       "      <td>1-65567-G-A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65567-G-A-1-65567</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>chr1:65567</td>\n",
       "      <td>[\"G\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65567\",\"G\",\"C\",\"1\",\"65567\"]</td>\n",
       "      <td>1-65567-G-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65567-G-C-1-65567</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>chr1:65567</td>\n",
       "      <td>[\"G\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>[\"1\",\"65567\",\"G\",\"T\",\"1\",\"65567\"]</td>\n",
       "      <td>1-65567-G-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65567-G-T-1-65567</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>chr1:65568</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant\"]</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65568\",\"A\",\"C\",\"1\",\"65568\"]</td>\n",
       "      <td>1-65568-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65568-A-C-1-65568</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>chr1:65568</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant\"]</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65568\",\"A\",\"G\",\"1\",\"65568\"]</td>\n",
       "      <td>1-65568-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65568-A-G-1-65568</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>chr1:65568</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"stop_gained\",\"HIGH\",\"OR4F5\",\"ENSG000001...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"stop_gained\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65568\",\"A\",\"T\",\"1\",\"65568\"]</td>\n",
       "      <td>1-65568-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65568-A-T-1-65568</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>chr1:65569</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"missense_variant\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65569\",\"A\",\"C\",\"1\",\"65569\"]</td>\n",
       "      <td>1-65569-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65569-A-C-1-65569</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>chr1:65569</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"missense_variant\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65569\",\"A\",\"G\",\"1\",\"65569\"]</td>\n",
       "      <td>1-65569-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65569-A-G-1-65569</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>chr1:65569</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant\"]</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65569\",\"A\",\"T\",\"1\",\"65569\"]</td>\n",
       "      <td>1-65569-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65569-A-T-1-65569</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>chr1:65570</td>\n",
       "      <td>[\"G\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"missense_variant\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65570\",\"G\",\"C\",\"1\",\"65570\"]</td>\n",
       "      <td>1-65570-G-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65570-G-C-1-65570</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>chr1:65570</td>\n",
       "      <td>[\"G\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"missense_variant\",\"downstream_gene_variant\"]</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65570\",\"G\",\"T\",\"1\",\"65570\"]</td>\n",
       "      <td>1-65570-G-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65570-G-T-1-65570</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>chr1:65571</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant&amp;s...</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65571\",\"A\",\"C\",\"1\",\"65571\"]</td>\n",
       "      <td>1-65571-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65571-A-C-1-65571</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>chr1:65571</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant&amp;s...</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65571\",\"A\",\"G\",\"1\",\"65571\"]</td>\n",
       "      <td>1-65571-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65571-A-G-1-65571</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>chr1:65571</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"stop_gained&amp;splice...</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65571\",\"A\",\"T\",\"1\",\"65571\"]</td>\n",
       "      <td>1-65571-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65571-A-T-1-65571</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>chr1:65572</td>\n",
       "      <td>[\"A\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"missense_variant&amp;splice_region_variant\"...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"missense_variant&amp;splice_region_variant\",\"dow...</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65572\",\"A\",\"C\",\"1\",\"65572\"]</td>\n",
       "      <td>1-65572-A-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65572-A-C-1-65572</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>chr1:65572</td>\n",
       "      <td>[\"A\",\"G\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"G\",\"G\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant&amp;s...</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65572\",\"A\",\"G\",\"1\",\"65572\"]</td>\n",
       "      <td>1-65572-A-G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65572-A-G-1-65572</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>chr1:65572</td>\n",
       "      <td>[\"A\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"missense_variant&amp;splice_region_variant\"...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"missense_variant&amp;splice_region_variant\",\"dow...</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65572\",\"A\",\"T\",\"1\",\"65572\"]</td>\n",
       "      <td>1-65572-A-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65572-A-T-1-65572</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>chr1:65573</td>\n",
       "      <td>[\"G\",\"C\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...</td>\n",
       "      <td>[\"C\",\"C\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"missense_variant&amp;s...</td>\n",
       "      <td>[\"MODIFIER\",\"MODERATE\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65573\",\"G\",\"C\",\"1\",\"65573\"]</td>\n",
       "      <td>1-65573-G-C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65573-G-C-1-65573</td>\n",
       "      <td>[1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>chr1:65573</td>\n",
       "      <td>[\"G\",\"T\"]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-10.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>{\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...</td>\n",
       "      <td>[[\"T\",\"missense_variant&amp;splice_region_variant\"...</td>\n",
       "      <td>[\"T\",\"T\"]</td>\n",
       "      <td>[\"missense_variant&amp;splice_region_variant\",\"dow...</td>\n",
       "      <td>[\"MODERATE\",\"MODIFIER\"]</td>\n",
       "      <td>...</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>[\"1\",\"65573\",\"G\",\"T\",\"1\",\"65573\"]</td>\n",
       "      <td>1-65573-G-T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1-65573-G-T-1-65573</td>\n",
       "      <td>[0]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>25 rows × 25 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         locus    alleles  rsid  qual  filters  \\\n",
       "0   chr1:65565  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "1   chr1:65565  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "2   chr1:65565  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "3   chr1:65566  [\"T\",\"A\"]   NaN -10.0      NaN   \n",
       "4   chr1:65566  [\"T\",\"C\"]   NaN -10.0      NaN   \n",
       "5   chr1:65566  [\"T\",\"G\"]   NaN -10.0      NaN   \n",
       "6   chr1:65567  [\"G\",\"A\"]   NaN -10.0      NaN   \n",
       "7   chr1:65567  [\"G\",\"C\"]   NaN -10.0      NaN   \n",
       "8   chr1:65567  [\"G\",\"T\"]   NaN -10.0      NaN   \n",
       "9   chr1:65568  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "10  chr1:65568  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "11  chr1:65568  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "12  chr1:65569  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "13  chr1:65569  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "14  chr1:65569  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "15  chr1:65570  [\"G\",\"C\"]   NaN -10.0      NaN   \n",
       "16  chr1:65570  [\"G\",\"T\"]   NaN -10.0      NaN   \n",
       "17  chr1:65571  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "18  chr1:65571  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "19  chr1:65571  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "20  chr1:65572  [\"A\",\"C\"]   NaN -10.0      NaN   \n",
       "21  chr1:65572  [\"A\",\"G\"]   NaN -10.0      NaN   \n",
       "22  chr1:65572  [\"A\",\"T\"]   NaN -10.0      NaN   \n",
       "23  chr1:65573  [\"G\",\"C\"]   NaN -10.0      NaN   \n",
       "24  chr1:65573  [\"G\",\"T\"]   NaN -10.0      NaN   \n",
       "\n",
       "                                                 info  \\\n",
       "0   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "1   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "2   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "3   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "4   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "5   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "6   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "7   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "8   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "9   {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "10  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "11  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "12  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "13  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "14  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "15  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "16  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "17  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "18  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "19  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "20  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "21  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "22  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "23  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "24  {\"NS\":null,\"DP\":null,\"AF\":null,\"AA\":null,\"DB\":...   \n",
       "\n",
       "                                            CSQ_split     Allele  \\\n",
       "0   [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "1   [[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"G\",\"G\"]   \n",
       "2   [[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"T\",\"T\"]   \n",
       "3   [[\"A\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"A\",\"A\"]   \n",
       "4   [[\"C\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...  [\"C\",\"C\"]   \n",
       "5   [[\"G\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...  [\"G\",\"G\"]   \n",
       "6   [[\"A\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...  [\"A\",\"A\"]   \n",
       "7   [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "8   [[\"T\",\"start_lost\",\"HIGH\",\"OR4F5\",\"ENSG0000018...  [\"T\",\"T\"]   \n",
       "9   [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "10  [[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"G\",\"G\"]   \n",
       "11  [[\"T\",\"stop_gained\",\"HIGH\",\"OR4F5\",\"ENSG000001...  [\"T\",\"T\"]   \n",
       "12  [[\"C\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...  [\"C\",\"C\"]   \n",
       "13  [[\"G\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...  [\"G\",\"G\"]   \n",
       "14  [[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"T\",\"T\"]   \n",
       "15  [[\"C\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...  [\"C\",\"C\"]   \n",
       "16  [[\"T\",\"missense_variant\",\"MODERATE\",\"OR4F5\",\"E...  [\"T\",\"T\"]   \n",
       "17  [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "18  [[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"G\",\"G\"]   \n",
       "19  [[\"T\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"T\",\"T\"]   \n",
       "20  [[\"C\",\"missense_variant&splice_region_variant\"...  [\"C\",\"C\"]   \n",
       "21  [[\"G\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"G\",\"G\"]   \n",
       "22  [[\"T\",\"missense_variant&splice_region_variant\"...  [\"T\",\"T\"]   \n",
       "23  [[\"C\",\"downstream_gene_variant\",\"MODIFIER\",\"OR...  [\"C\",\"C\"]   \n",
       "24  [[\"T\",\"missense_variant&splice_region_variant\"...  [\"T\",\"T\"]   \n",
       "\n",
       "                                          Consequence  \\\n",
       "0            [\"downstream_gene_variant\",\"start_lost\"]   \n",
       "1            [\"downstream_gene_variant\",\"start_lost\"]   \n",
       "2            [\"downstream_gene_variant\",\"start_lost\"]   \n",
       "3            [\"downstream_gene_variant\",\"start_lost\"]   \n",
       "4            [\"start_lost\",\"downstream_gene_variant\"]   \n",
       "5            [\"start_lost\",\"downstream_gene_variant\"]   \n",
       "6            [\"start_lost\",\"downstream_gene_variant\"]   \n",
       "7            [\"downstream_gene_variant\",\"start_lost\"]   \n",
       "8            [\"start_lost\",\"downstream_gene_variant\"]   \n",
       "9      [\"downstream_gene_variant\",\"missense_variant\"]   \n",
       "10     [\"downstream_gene_variant\",\"missense_variant\"]   \n",
       "11          [\"stop_gained\",\"downstream_gene_variant\"]   \n",
       "12     [\"missense_variant\",\"downstream_gene_variant\"]   \n",
       "13     [\"missense_variant\",\"downstream_gene_variant\"]   \n",
       "14     [\"downstream_gene_variant\",\"missense_variant\"]   \n",
       "15     [\"missense_variant\",\"downstream_gene_variant\"]   \n",
       "16     [\"missense_variant\",\"downstream_gene_variant\"]   \n",
       "17  [\"downstream_gene_variant\",\"missense_variant&s...   \n",
       "18  [\"downstream_gene_variant\",\"missense_variant&s...   \n",
       "19  [\"downstream_gene_variant\",\"stop_gained&splice...   \n",
       "20  [\"missense_variant&splice_region_variant\",\"dow...   \n",
       "21  [\"downstream_gene_variant\",\"missense_variant&s...   \n",
       "22  [\"missense_variant&splice_region_variant\",\"dow...   \n",
       "23  [\"downstream_gene_variant\",\"missense_variant&s...   \n",
       "24  [\"missense_variant&splice_region_variant\",\"dow...   \n",
       "\n",
       "                     IMPACT  ... STOPLOST STARTLOST  \\\n",
       "0       [\"MODIFIER\",\"HIGH\"]  ...    False      True   \n",
       "1       [\"MODIFIER\",\"HIGH\"]  ...    False      True   \n",
       "2       [\"MODIFIER\",\"HIGH\"]  ...    False      True   \n",
       "3       [\"MODIFIER\",\"HIGH\"]  ...    False      True   \n",
       "4       [\"HIGH\",\"MODIFIER\"]  ...    False      True   \n",
       "5       [\"HIGH\",\"MODIFIER\"]  ...    False      True   \n",
       "6       [\"HIGH\",\"MODIFIER\"]  ...    False      True   \n",
       "7       [\"MODIFIER\",\"HIGH\"]  ...    False      True   \n",
       "8       [\"HIGH\",\"MODIFIER\"]  ...    False      True   \n",
       "9   [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "10  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "11      [\"HIGH\",\"MODIFIER\"]  ...    False     False   \n",
       "12  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "13  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "14  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "15  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "16  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "17  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "18  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "19      [\"MODIFIER\",\"HIGH\"]  ...    False     False   \n",
       "20  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "21  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "22  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "23  [\"MODIFIER\",\"MODERATE\"]  ...    False     False   \n",
       "24  [\"MODERATE\",\"MODIFIER\"]  ...    False     False   \n",
       "\n",
       "                               ID_raw           ID  MISTIC  CCR  CHROM  \\\n",
       "0   [\"1\",\"65565\",\"A\",\"C\",\"1\",\"65565\"]  1-65565-A-C     NaN  NaN      1   \n",
       "1   [\"1\",\"65565\",\"A\",\"G\",\"1\",\"65565\"]  1-65565-A-G     NaN  NaN      1   \n",
       "2   [\"1\",\"65565\",\"A\",\"T\",\"1\",\"65565\"]  1-65565-A-T     NaN  NaN      1   \n",
       "3   [\"1\",\"65566\",\"T\",\"A\",\"1\",\"65566\"]  1-65566-T-A     NaN  NaN      1   \n",
       "4   [\"1\",\"65566\",\"T\",\"C\",\"1\",\"65566\"]  1-65566-T-C     NaN  NaN      1   \n",
       "5   [\"1\",\"65566\",\"T\",\"G\",\"1\",\"65566\"]  1-65566-T-G     NaN  NaN      1   \n",
       "6   [\"1\",\"65567\",\"G\",\"A\",\"1\",\"65567\"]  1-65567-G-A     NaN  NaN      1   \n",
       "7   [\"1\",\"65567\",\"G\",\"C\",\"1\",\"65567\"]  1-65567-G-C     NaN  NaN      1   \n",
       "8   [\"1\",\"65567\",\"G\",\"T\",\"1\",\"65567\"]  1-65567-G-T     NaN  NaN      1   \n",
       "9   [\"1\",\"65568\",\"A\",\"C\",\"1\",\"65568\"]  1-65568-A-C     NaN  NaN      1   \n",
       "10  [\"1\",\"65568\",\"A\",\"G\",\"1\",\"65568\"]  1-65568-A-G     NaN  NaN      1   \n",
       "11  [\"1\",\"65568\",\"A\",\"T\",\"1\",\"65568\"]  1-65568-A-T     NaN  NaN      1   \n",
       "12  [\"1\",\"65569\",\"A\",\"C\",\"1\",\"65569\"]  1-65569-A-C     NaN  NaN      1   \n",
       "13  [\"1\",\"65569\",\"A\",\"G\",\"1\",\"65569\"]  1-65569-A-G     NaN  NaN      1   \n",
       "14  [\"1\",\"65569\",\"A\",\"T\",\"1\",\"65569\"]  1-65569-A-T     NaN  NaN      1   \n",
       "15  [\"1\",\"65570\",\"G\",\"C\",\"1\",\"65570\"]  1-65570-G-C     NaN  NaN      1   \n",
       "16  [\"1\",\"65570\",\"G\",\"T\",\"1\",\"65570\"]  1-65570-G-T     NaN  NaN      1   \n",
       "17  [\"1\",\"65571\",\"A\",\"C\",\"1\",\"65571\"]  1-65571-A-C     NaN  NaN      1   \n",
       "18  [\"1\",\"65571\",\"A\",\"G\",\"1\",\"65571\"]  1-65571-A-G     NaN  NaN      1   \n",
       "19  [\"1\",\"65571\",\"A\",\"T\",\"1\",\"65571\"]  1-65571-A-T     NaN  NaN      1   \n",
       "20  [\"1\",\"65572\",\"A\",\"C\",\"1\",\"65572\"]  1-65572-A-C     NaN  NaN      1   \n",
       "21  [\"1\",\"65572\",\"A\",\"G\",\"1\",\"65572\"]  1-65572-A-G     NaN  NaN      1   \n",
       "22  [\"1\",\"65572\",\"A\",\"T\",\"1\",\"65572\"]  1-65572-A-T     NaN  NaN      1   \n",
       "23  [\"1\",\"65573\",\"G\",\"C\",\"1\",\"65573\"]  1-65573-G-C     NaN  NaN      1   \n",
       "24  [\"1\",\"65573\",\"G\",\"T\",\"1\",\"65573\"]  1-65573-G-T     NaN  NaN      1   \n",
       "\n",
       "   CHROM_37              FULL_ID  Missense_index  \n",
       "0         1  1-65565-A-C-1-65565              []  \n",
       "1         1  1-65565-A-G-1-65565              []  \n",
       "2         1  1-65565-A-T-1-65565              []  \n",
       "3         1  1-65566-T-A-1-65566              []  \n",
       "4         1  1-65566-T-C-1-65566              []  \n",
       "5         1  1-65566-T-G-1-65566              []  \n",
       "6         1  1-65567-G-A-1-65567              []  \n",
       "7         1  1-65567-G-C-1-65567              []  \n",
       "8         1  1-65567-G-T-1-65567              []  \n",
       "9         1  1-65568-A-C-1-65568             [1]  \n",
       "10        1  1-65568-A-G-1-65568             [1]  \n",
       "11        1  1-65568-A-T-1-65568              []  \n",
       "12        1  1-65569-A-C-1-65569             [0]  \n",
       "13        1  1-65569-A-G-1-65569             [0]  \n",
       "14        1  1-65569-A-T-1-65569             [1]  \n",
       "15        1  1-65570-G-C-1-65570             [0]  \n",
       "16        1  1-65570-G-T-1-65570             [0]  \n",
       "17        1  1-65571-A-C-1-65571             [1]  \n",
       "18        1  1-65571-A-G-1-65571             [1]  \n",
       "19        1  1-65571-A-T-1-65571              []  \n",
       "20        1  1-65572-A-C-1-65572             [0]  \n",
       "21        1  1-65572-A-G-1-65572             [1]  \n",
       "22        1  1-65572-A-T-1-65572             [0]  \n",
       "23        1  1-65573-G-C-1-65573             [1]  \n",
       "24        1  1-65573-G-T-1-65573             [0]  \n",
       "\n",
       "[25 rows x 25 columns]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dbnsfp_vcf_hail_pandas.head(25)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "cab926fb-f237-45d8-86bf-d14ed5659e19",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "2\n",
      "3\n",
      "4\n",
      "5\n",
      "6\n",
      "7\n",
      "8\n",
      "9\n",
      "10\n",
      "11\n",
      "12\n",
      "13\n",
      "14\n",
      "15\n",
      "16\n",
      "17\n",
      "18\n",
      "19\n",
      "20\n",
      "21\n",
      "22\n",
      "X\n",
      "Y\n",
      "M\n"
     ]
    }
   ],
   "source": [
    "# Split into chunks for each chr\n",
    "\n",
    "for chromosome in dbnsfp_vcf_hail_pandas.CHROM.unique():\n",
    "    print(chromosome)\n",
    "    dbnsfp_vcf_hail_pandas.loc[dbnsfp_vcf_hail_pandas['CHROM'] == chromosome].to_parquet('/gstock/MISTIC/CAGI6/dbnsfp_vep_mistic_chr{}.parquet'.format(str(chromosome)), index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "91a4ce48-1dd0-4881-a8cb-34508983c02e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#chr</th>\n",
       "      <th>pos(1-based)</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>score</th>\n",
       "      <th>sd</th>\n",
       "      <th>pred</th>\n",
       "      <th>comments</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>Consequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>65565</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>0</td>\n",
       "      <td>U</td>\n",
       "      <td>No missense in VEP predictions / HIGH impact</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>65565</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>0</td>\n",
       "      <td>U</td>\n",
       "      <td>No missense in VEP predictions / HIGH impact</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>65565</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>0</td>\n",
       "      <td>U</td>\n",
       "      <td>No missense in VEP predictions / HIGH impact</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>65566</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>0</td>\n",
       "      <td>U</td>\n",
       "      <td>No missense in VEP predictions / HIGH impact</td>\n",
       "      <td>[\"MODIFIER\",\"HIGH\"]</td>\n",
       "      <td>[\"downstream_gene_variant\",\"start_lost\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>65566</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>0</td>\n",
       "      <td>U</td>\n",
       "      <td>No missense in VEP predictions / HIGH impact</td>\n",
       "      <td>[\"HIGH\",\"MODIFIER\"]</td>\n",
       "      <td>[\"start_lost\",\"downstream_gene_variant\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>1</td>\n",
       "      <td>69239</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>0.31938</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>*</td>\n",
       "      <td>[\"MODERATE\"]</td>\n",
       "      <td>[\"missense_variant\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>1</td>\n",
       "      <td>69241</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>0.51261</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>*</td>\n",
       "      <td>[\"MODERATE\"]</td>\n",
       "      <td>[\"missense_variant\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>1</td>\n",
       "      <td>69241</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>0.23673</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>*</td>\n",
       "      <td>[\"MODERATE\"]</td>\n",
       "      <td>[\"missense_variant\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>1</td>\n",
       "      <td>69241</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>0.20214</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>*</td>\n",
       "      <td>[\"MODERATE\"]</td>\n",
       "      <td>[\"missense_variant\"]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>1</td>\n",
       "      <td>69242</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>0.39765</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>*</td>\n",
       "      <td>[\"MODERATE\"]</td>\n",
       "      <td>[\"missense_variant\"]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>500 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    #chr pos(1-based) ref alt    score  sd pred  \\\n",
       "0      1        65565   A   C  1.00000   0    U   \n",
       "1      1        65565   A   G  1.00000   0    U   \n",
       "2      1        65565   A   T  1.00000   0    U   \n",
       "3      1        65566   T   A  1.00000   0    U   \n",
       "4      1        65566   T   C  1.00000   0    U   \n",
       "..   ...          ...  ..  ..      ...  ..  ...   \n",
       "495    1        69239   C   T  0.31938   0    T   \n",
       "496    1        69241   C   A  0.51261   0    D   \n",
       "497    1        69241   C   G  0.23673   0    T   \n",
       "498    1        69241   C   T  0.20214   0    T   \n",
       "499    1        69242   C   A  0.39765   0    T   \n",
       "\n",
       "                                         comments               IMPACT  \\\n",
       "0    No missense in VEP predictions / HIGH impact  [\"MODIFIER\",\"HIGH\"]   \n",
       "1    No missense in VEP predictions / HIGH impact  [\"MODIFIER\",\"HIGH\"]   \n",
       "2    No missense in VEP predictions / HIGH impact  [\"MODIFIER\",\"HIGH\"]   \n",
       "3    No missense in VEP predictions / HIGH impact  [\"MODIFIER\",\"HIGH\"]   \n",
       "4    No missense in VEP predictions / HIGH impact  [\"HIGH\",\"MODIFIER\"]   \n",
       "..                                            ...                  ...   \n",
       "495                                             *         [\"MODERATE\"]   \n",
       "496                                             *         [\"MODERATE\"]   \n",
       "497                                             *         [\"MODERATE\"]   \n",
       "498                                             *         [\"MODERATE\"]   \n",
       "499                                             *         [\"MODERATE\"]   \n",
       "\n",
       "                                  Consequence  \n",
       "0    [\"downstream_gene_variant\",\"start_lost\"]  \n",
       "1    [\"downstream_gene_variant\",\"start_lost\"]  \n",
       "2    [\"downstream_gene_variant\",\"start_lost\"]  \n",
       "3    [\"downstream_gene_variant\",\"start_lost\"]  \n",
       "4    [\"start_lost\",\"downstream_gene_variant\"]  \n",
       "..                                        ...  \n",
       "495                      [\"missense_variant\"]  \n",
       "496                      [\"missense_variant\"]  \n",
       "497                      [\"missense_variant\"]  \n",
       "498                      [\"missense_variant\"]  \n",
       "499                      [\"missense_variant\"]  \n",
       "\n",
       "[500 rows x 10 columns]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test = dbnsfp_vcf_hail_pandas.head(500)[['locus','alleles','MISTIC', 'IMPACT', 'Consequence']]\n",
    "\n",
    "test['#chr'] = test['locus'].apply(lambda r: r.split(':')[0].replace('chr',''))\n",
    "test['#chr'] = test['#chr'].astype(str)\n",
    "test['pos(1-based)'] = test['locus'].apply(lambda r: r.split(':')[1])\n",
    "test['ref'] = test['alleles'].apply(lambda r: eval(r)[0])\n",
    "test['alt'] = test['alleles'].apply(lambda r: eval(r)[1])\n",
    "test = test.rename({'MISTIC' : 'score'}, axis=1)\n",
    "test.loc[(test['IMPACT'].str.contains('HIGH')) & (test['score'].isna() == True), 'comments'] = 'No missense in VEP predictions / HIGH impact'\n",
    "test.loc[(test['IMPACT'].str.contains('LOW|MODIFIER')) & (test['score'].isna() == True) & (test['comments'].isna() == True), 'comments'] = 'No missense in VEP predictions / LOW|MODIFIER impact'\n",
    "\n",
    "\n",
    "test.loc[test['score'] >= 0.5, 'pred'] = 'D'\n",
    "test.loc[test['score'] < 0.5, 'pred'] = 'T'\n",
    "test.loc[test['score'].isna() == True, 'pred'] = 'U'\n",
    "\n",
    "\n",
    "test.loc[(test['IMPACT'].str.contains('HIGH')) & (test['score'].isna() == True) , 'score'] = 1\n",
    "test.loc[(test['IMPACT'].str.contains('LOW|MODIFIER')) & (test['score'].isna() == True) , 'score'] = 0\n",
    "\n",
    "\n",
    "test['sd'] = 0\n",
    "test['comments'] = test['comments'].fillna('*')\n",
    "test\n",
    "test[['#chr', 'pos(1-based)','ref','alt','score','sd','pred', 'comments']].to_csv('/gstock/MISTIC/CAGI6/download/TEST/test.csv', sep='\\t', index=False)\n",
    "test[['#chr', 'pos(1-based)','ref','alt','score','sd','pred', 'comments', 'IMPACT', 'Consequence']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "7ea2581b-1cc1-47b5-9d77-5df471933f0b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['gnomAD_exomes_AF',\n",
       " 'gnomAD_exomes_AFR_AF',\n",
       " 'gnomAD_exomes_AMR_AF',\n",
       " 'gnomAD_exomes_EAS_AF',\n",
       " 'gnomAD_exomes_NFE_AF',\n",
       " 'gnomAD_exomes_SAS_AF']"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gnomad_cols = [e for e in feature_cols if 'gnomAD' in e ]\n",
    "gnomad_cols"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 173,
   "id": "ec30724f-34ca-4c61-a42c-523b853d5fb7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>Gene</th>\n",
       "      <th>...</th>\n",
       "      <th>clinvar_OMIM_id</th>\n",
       "      <th>clinvar_Orphanet_id</th>\n",
       "      <th>Interpro_domain</th>\n",
       "      <th>GTEx_V8_gene</th>\n",
       "      <th>GTEx_V8_tissue</th>\n",
       "      <th>Geuvadis_eQTL_target_gene</th>\n",
       "      <th>MISTIC_score</th>\n",
       "      <th>MISTIC_sd</th>\n",
       "      <th>MISTIC_pred</th>\n",
       "      <th>MISTIC_comments</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>21</td>\n",
       "      <td>27012134</td>\n",
       "      <td>21_27012134_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=818128;CLNDISDB=MONDO:MONDO:0032938,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>JAM2</td>\n",
       "      <td>...</td>\n",
       "      <td>618824</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.242621</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>21</td>\n",
       "      <td>27066149</td>\n",
       "      <td>21_27066149_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=818131;CLNDISDB=MONDO:MONDO:0032938,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>JAM2</td>\n",
       "      <td>...</td>\n",
       "      <td>618824</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Immunoglobulin V-set domain|Immunoglobulin-l...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.641090</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>21</td>\n",
       "      <td>27071098</td>\n",
       "      <td>21_27071098_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=818129;CLNDISDB=MONDO:MONDO:0032938,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>JAM2</td>\n",
       "      <td>...</td>\n",
       "      <td>618824</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Immunoglobulin-like domain|Immunoglobulin su...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.933320</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>W/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>21</td>\n",
       "      <td>27264120</td>\n",
       "      <td>21_27264120_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=626283;CLNDISDB=MONDO:MONDO:0005620,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>APP</td>\n",
       "      <td>...</td>\n",
       "      <td>605714</td>\n",
       "      <td>ORPHA85458</td>\n",
       "      <td>Amyloidogenic glycoprotein, amyloid-beta pepti...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.799610</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>21</td>\n",
       "      <td>28213391</td>\n",
       "      <td>21_28213391_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=918095;CLNDISDB=MONDO:MONDO:0005387,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>ADAMTS1</td>\n",
       "      <td>...</td>\n",
       "      <td>PS311360</td>\n",
       "      <td>.</td>\n",
       "      <td>Peptidase M12B, ADAM/reprolysin|Peptidase M12B...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.152990</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>S/Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>21</td>\n",
       "      <td>33038797</td>\n",
       "      <td>21_33038797_T_C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=861431;CLNDISDB=MONDO:MONDO:0007103,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>105400</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.888600</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>S/P</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>21</td>\n",
       "      <td>33039674</td>\n",
       "      <td>21_33039674_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=920983;CLNDISDB=MedGen:CN517202;CLNDN...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.913210</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>21</td>\n",
       "      <td>33040802</td>\n",
       "      <td>21_33040802_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=861434;CLNDISDB=MONDO:MONDO:0007103,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>105400</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.928640</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>21</td>\n",
       "      <td>33040803</td>\n",
       "      <td>21_33040803_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=961894;CLNDISDB=MONDO:MONDO:0007103,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>105400</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.939960</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>21</td>\n",
       "      <td>33040822</td>\n",
       "      <td>21_33040822_T_G</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1052613;CLNDISDB=MONDO:MONDO:0007103,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.738720</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>N/K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>21</td>\n",
       "      <td>33040826</td>\n",
       "      <td>21_33040826_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=861435;CLNDISDB=MONDO:MONDO:0007103,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>105400</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.926230</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>E/K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>21</td>\n",
       "      <td>33040839</td>\n",
       "      <td>21_33040839_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1152912;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>SOD1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Superoxide dismutase, copper/zinc binding doma...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.843900</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>T/I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>21</td>\n",
       "      <td>34775850</td>\n",
       "      <td>21_34775850_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1056638;CLNDISDB=MONDO:MONDO:0013953,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>IFNGR2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.165504</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>21</td>\n",
       "      <td>34775850</td>\n",
       "      <td>21_34775850_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=975708;CLNDISDB=MONDO:MONDO:0013953,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>IFNGR2</td>\n",
       "      <td>...</td>\n",
       "      <td>614889</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.154573</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>21</td>\n",
       "      <td>34889711</td>\n",
       "      <td>21_34889711_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=904905;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>GART</td>\n",
       "      <td>...</td>\n",
       "      <td>189960</td>\n",
       "      <td>ORPHA1199</td>\n",
       "      <td>PurM-like, C-terminal domain;PurM-like, C-term...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.390680</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>L/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>21</td>\n",
       "      <td>34945648</td>\n",
       "      <td>21_34945648_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=815961;CLNDISDB=MONDO:MONDO:0014936,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>SON</td>\n",
       "      <td>...</td>\n",
       "      <td>617140</td>\n",
       "      <td>ORPHA500150</td>\n",
       "      <td>G-patch domain|G-patch domain|G-patch domain;G...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.713870</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>21</td>\n",
       "      <td>34950680</td>\n",
       "      <td>21_34950680_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.233880</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>P/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>21</td>\n",
       "      <td>34950680</td>\n",
       "      <td>21_34950680_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.233880</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>R/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>21</td>\n",
       "      <td>34950680</td>\n",
       "      <td>21_34950680_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.233880</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>P/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>21</td>\n",
       "      <td>34950680</td>\n",
       "      <td>21_34950680_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.233880</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>R/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>21</td>\n",
       "      <td>34955949</td>\n",
       "      <td>21_34955949_T_C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=999978;CLNDISDB=.|MONDO:MONDO:0016817...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.654430</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>Y/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>34957011</td>\n",
       "      <td>21_34957011_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=999979;CLNDISDB=.|MONDO:MONDO:0016817...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.721110</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>21</td>\n",
       "      <td>34957050</td>\n",
       "      <td>21_34957050_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162254;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.679830</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>21</td>\n",
       "      <td>34958396</td>\n",
       "      <td>21_34958396_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1162256;CLNDISDB=MONDO:MONDO:0016817,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DONSON</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.495180</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>F/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>21</td>\n",
       "      <td>35254586</td>\n",
       "      <td>21_35254586_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=904906;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>ITSN1</td>\n",
       "      <td>...</td>\n",
       "      <td>189960</td>\n",
       "      <td>ORPHA1199</td>\n",
       "      <td>Pleckstrin homology domain;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.914020</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>21</td>\n",
       "      <td>35821932</td>\n",
       "      <td>21_35821932_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1056640;CLNDISDB=MONDO:MONDO:0002442,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>KCNE1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.132754</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>21</td>\n",
       "      <td>35821932</td>\n",
       "      <td>21_35821932_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1056640;CLNDISDB=MONDO:MONDO:0002442,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>KCNE1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.;.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.405664</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>21</td>\n",
       "      <td>36164605</td>\n",
       "      <td>21_36164605_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976801;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Runx, C-terminal  domain;.;.;Runx, C-terminal ...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.218180</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>S/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>21</td>\n",
       "      <td>36164619</td>\n",
       "      <td>21_36164619_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976803;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Runx, C-terminal  domain;.;.;Runx, C-terminal ...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.099688</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>V/G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>21</td>\n",
       "      <td>36231791</td>\n",
       "      <td>21_36231791_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=615635;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>601399</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.984310</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>21</td>\n",
       "      <td>36231798</td>\n",
       "      <td>21_36231798_T_C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976804;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.984580</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>T/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>21</td>\n",
       "      <td>36252856</td>\n",
       "      <td>21_36252856_C_A</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=857423;CLNDISDB=MONDO:MONDO:0100083,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>601399</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.976880</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>21</td>\n",
       "      <td>36252959</td>\n",
       "      <td>21_36252959_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976806;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.968070</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>21</td>\n",
       "      <td>36259175</td>\n",
       "      <td>21_36259175_A_T</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=430422;CLNDISDB=MONDO:MONDO:0100083,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>601399</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.967160</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>W/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>21</td>\n",
       "      <td>36259176</td>\n",
       "      <td>21_36259176_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=653831;CLNDISDB=MONDO:MONDO:0011071,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>601399</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.909230</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>H/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>21</td>\n",
       "      <td>36259177</td>\n",
       "      <td>21_36259177_T_G</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=552387;CLNDISDB=MONDO:MONDO:0100083,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>601399</td>\n",
       "      <td>.</td>\n",
       "      <td>Runt domain|Runt domain;.;.;Runt domain|Runt d...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.953190</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>H/P</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>21</td>\n",
       "      <td>36259394</td>\n",
       "      <td>21_36259394_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976809;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.906094</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>21</td>\n",
       "      <td>36259394</td>\n",
       "      <td>21_36259394_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=976809;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.906094</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>21</td>\n",
       "      <td>37833803</td>\n",
       "      <td>21_37833803_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590337;CLNDISDB=MONDO:MONDO:0013537,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CLDN14</td>\n",
       "      <td>...</td>\n",
       "      <td>614035</td>\n",
       "      <td>.</td>\n",
       "      <td>Claudin, conserved site;Claudin, conserved sit...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.903510</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>C/Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>21</td>\n",
       "      <td>38126584</td>\n",
       "      <td>21_38126584_T_G</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>AF_EXAC=0.00003;ALLELEID=792007;CLNDISDB=MONDO...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>HLCS</td>\n",
       "      <td>...</td>\n",
       "      <td>253270</td>\n",
       "      <td>ORPHA79242</td>\n",
       "      <td>Biotin protein ligase, C-terminal;Biotin prote...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.751810</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>21</td>\n",
       "      <td>38853123</td>\n",
       "      <td>21_38853123_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=648994;CLNDISDB=MONDO:MONDO:0013578,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DYRK1A</td>\n",
       "      <td>...</td>\n",
       "      <td>614104</td>\n",
       "      <td>ORPHA464306</td>\n",
       "      <td>Protein kinase domain|Protein kinase, ATP bind...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.971090</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>21</td>\n",
       "      <td>38862726</td>\n",
       "      <td>21_38862726_T_G</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1151744;CLNDISDB=Human_Phenotype_Onto...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DYRK1A</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Protein kinase domain|Protein kinase domain|Pr...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.895650</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>I/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>21</td>\n",
       "      <td>38865339</td>\n",
       "      <td>21_38865339_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=615979;CLNDISDB=MONDO:MONDO:0013578,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DYRK1A</td>\n",
       "      <td>...</td>\n",
       "      <td>614104</td>\n",
       "      <td>ORPHA464306</td>\n",
       "      <td>Protein kinase domain|Protein kinase domain|Pr...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.962630</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>S/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>21</td>\n",
       "      <td>38865409</td>\n",
       "      <td>21_38865409_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=920984;CLNDISDB=MedGen:CN517202;CLNDN...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>DYRK1A</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Protein kinase domain|Protein kinase domain|Pr...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.958940</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/W</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>21</td>\n",
       "      <td>38877608</td>\n",
       "      <td>21_38877608_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=951237;CLNDISDB=MONDO:MONDO:0013578,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DYRK1A</td>\n",
       "      <td>...</td>\n",
       "      <td>614104</td>\n",
       "      <td>ORPHA464306</td>\n",
       "      <td>Protein kinase domain|Protein kinase domain|Pr...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.761110</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>21</td>\n",
       "      <td>40570769</td>\n",
       "      <td>21_40570769_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=970556;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>BRWD1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.118330</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>Q/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>21</td>\n",
       "      <td>40665912</td>\n",
       "      <td>21_40665912_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=918097;CLNDISDB=MONDO:MONDO:0005387,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>BRWD1</td>\n",
       "      <td>...</td>\n",
       "      <td>PS311360</td>\n",
       "      <td>.</td>\n",
       "      <td>WD40-repeat-containing domain;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.216200</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>R/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>21</td>\n",
       "      <td>41455850</td>\n",
       "      <td>21_41455850_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1087103;CLNDISDB=.;CLNDN=See_cases;CL...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DSCAM</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Fibronectin type III|Fibronectin type III|Fibr...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.492790</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>G/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>21</td>\n",
       "      <td>41561159</td>\n",
       "      <td>21_41561159_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590759;CLNDISDB=MONDO:MONDO:0018309,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>DSCAM</td>\n",
       "      <td>...</td>\n",
       "      <td>PS142623</td>\n",
       "      <td>ORPHA388</td>\n",
       "      <td>Immunoglobulin-like domain;Immunoglobulin-like...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.247440</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>A/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>21</td>\n",
       "      <td>43231165</td>\n",
       "      <td>21_43231165_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=860689;CLNDISDB=MedGen:CN517202;CLNDN...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>PRDM15</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;Zinc finger C2H2-type|Zinc finger C2H2-typ...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.965100</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>C/Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>21</td>\n",
       "      <td>43800309</td>\n",
       "      <td>21_43800309_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1053403;CLNDISDB=Human_Phenotype_Onto...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>TMPRSS3</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Serine proteases, trypsin domain|Serine prot...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.821610</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>21</td>\n",
       "      <td>43803153</td>\n",
       "      <td>21_43803153_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=962918;CLNDISDB=MONDO:MONDO:0010987,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>TMPRSS3</td>\n",
       "      <td>...</td>\n",
       "      <td>601072</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Serine proteases, trypsin domain|Serine prot...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.941820</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>H/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>21</td>\n",
       "      <td>43809053</td>\n",
       "      <td>21_43809053_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1053405;CLNDISDB=Human_Phenotype_Onto...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>TMPRSS3</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Low-density lipoprotein (LDL) receptor class...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.921870</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>21</td>\n",
       "      <td>44282449</td>\n",
       "      <td>21_44282449_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=610999;CLNDISDB=MONDO:MONDO:0032691,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>WDR4</td>\n",
       "      <td>...</td>\n",
       "      <td>618347</td>\n",
       "      <td>.</td>\n",
       "      <td>WD40-repeat-containing domain;WD40-repeat-cont...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.680960</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>21</td>\n",
       "      <td>44483076</td>\n",
       "      <td>21_44483076_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=966848;CLNDISDB=MONDO:MONDO:0009352,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.866630</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>V/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>21</td>\n",
       "      <td>44483076</td>\n",
       "      <td>21_44483076_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=966848;CLNDISDB=MONDO:MONDO:0009352,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>236200</td>\n",
       "      <td>ORPHA394</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.866630</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>V/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21_44483148_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.805350</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21_44483148_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.805350</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>21</td>\n",
       "      <td>44485800</td>\n",
       "      <td>21_44485800_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>AF_EXAC=0.00004;ALLELEID=210437;CLNDISDB=Human...</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.955000</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>21</td>\n",
       "      <td>44485800</td>\n",
       "      <td>21_44485800_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>AF_EXAC=0.00004;ALLELEID=210437;CLNDISDB=Human...</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>236200</td>\n",
       "      <td>ORPHA394</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.955000</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>21</td>\n",
       "      <td>44486353</td>\n",
       "      <td>21_44486353_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848818;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.865950</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>21</td>\n",
       "      <td>44486353</td>\n",
       "      <td>21_44486353_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848818;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>236200</td>\n",
       "      <td>ORPHA394</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.865950</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>21</td>\n",
       "      <td>44488651</td>\n",
       "      <td>21_44488651_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=471673;CLNDISDB=MONDO:MONDO:0009352,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.673380</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>I/T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>21</td>\n",
       "      <td>44488651</td>\n",
       "      <td>21_44488651_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=471673;CLNDISDB=MONDO:MONDO:0009352,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>236200</td>\n",
       "      <td>ORPHA394</td>\n",
       "      <td>Pyridoxal-phosphate dependent enzyme;Pyridoxal...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.673380</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>I/T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>21</td>\n",
       "      <td>45050277</td>\n",
       "      <td>21_45050277_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1035760;CLNDISDB=MONDO:MONDO:0030985,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>HSF2BP</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.340330</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>S/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>21</td>\n",
       "      <td>45161630</td>\n",
       "      <td>21_45161630_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=800762;CLNDISDB=MONDO:MONDO:0032792,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>PDXK</td>\n",
       "      <td>...</td>\n",
       "      <td>618511</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.564970</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>N/K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>21</td>\n",
       "      <td>45173500</td>\n",
       "      <td>21_45173500_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>AF_EXAC=0.00001;ALLELEID=624077;CLNDISDB=MONDO...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>PDXK</td>\n",
       "      <td>...</td>\n",
       "      <td>618511</td>\n",
       "      <td>.</td>\n",
       "      <td>.;Pyridoxamine kinase/Phosphomethylpyrimidine ...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.228580</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>R/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>21</td>\n",
       "      <td>45509730</td>\n",
       "      <td>21_45509730_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=984452;CLNDISDB=Human_Phenotype_Ontol...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>TRAPPC10</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.551070</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>21</td>\n",
       "      <td>45705914</td>\n",
       "      <td>21_45705914_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=961986;CLNDISDB=MONDO:MONDO:0009411,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>AIRE</td>\n",
       "      <td>...</td>\n",
       "      <td>240300</td>\n",
       "      <td>ORPHA3453</td>\n",
       "      <td>HSR domain|HSR domain</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.722000</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/W</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>21</td>\n",
       "      <td>45705933</td>\n",
       "      <td>21_45705933_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=980969;CLNDISDB=MONDO:MONDO:0009411,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>AIRE</td>\n",
       "      <td>...</td>\n",
       "      <td>240300</td>\n",
       "      <td>ORPHA3453</td>\n",
       "      <td>HSR domain|HSR domain</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.934830</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>21</td>\n",
       "      <td>45752925</td>\n",
       "      <td>21_45752925_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=800625;CLNDISDB=.;CLNDN=Cone-rod_dege...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CFAP410</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.499090</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>D/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>21</td>\n",
       "      <td>46595995</td>\n",
       "      <td>21_46595995_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=858619;CLNDISDB=MONDO:MONDO:0030025,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>ADARB1</td>\n",
       "      <td>...</td>\n",
       "      <td>618862</td>\n",
       "      <td>.</td>\n",
       "      <td>Double-stranded RNA-binding domain|Double-stra...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.910570</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>K/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>21</td>\n",
       "      <td>46604933</td>\n",
       "      <td>21_46604933_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=858618;CLNDISDB=MONDO:MONDO:0030025,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>ADARB1</td>\n",
       "      <td>...</td>\n",
       "      <td>618862</td>\n",
       "      <td>.</td>\n",
       "      <td>Adenosine deaminase/editase|Adenosine deaminas...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.263620</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>S/G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>21</td>\n",
       "      <td>46624592</td>\n",
       "      <td>21_46624592_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=858620;CLNDISDB=MONDO:MONDO:0030025,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>ADARB1</td>\n",
       "      <td>...</td>\n",
       "      <td>618862</td>\n",
       "      <td>.</td>\n",
       "      <td>Adenosine deaminase/editase|Adenosine deaminas...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.776460</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>21</td>\n",
       "      <td>46642051</td>\n",
       "      <td>21_46642051_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=858621;CLNDISDB=MONDO:MONDO:0030025,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>ADARB1</td>\n",
       "      <td>...</td>\n",
       "      <td>618862</td>\n",
       "      <td>.</td>\n",
       "      <td>Adenosine deaminase/editase|Adenosine deaminas...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.651410</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>A/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>21</td>\n",
       "      <td>47408999</td>\n",
       "      <td>21_47408999_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=860694;CLNDISDB=MedGen:CN517202;CLNDN...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.776730</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>21</td>\n",
       "      <td>47409523</td>\n",
       "      <td>21_47409523_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848918;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.875210</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>21</td>\n",
       "      <td>47409540</td>\n",
       "      <td>21_47409540_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1064923;CLNDISDB=MONDO:MONDO:0024530,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.877090</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>21</td>\n",
       "      <td>47409541</td>\n",
       "      <td>21_47409541_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=649076;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.877050</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>21</td>\n",
       "      <td>47409541</td>\n",
       "      <td>21_47409541_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=929366;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.906860</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>21</td>\n",
       "      <td>47417635</td>\n",
       "      <td>21_47417635_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=608972;CLNDISDB=MONDO:MONDO:0009681,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A1</td>\n",
       "      <td>...</td>\n",
       "      <td>254090</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.677970</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>P/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>21</td>\n",
       "      <td>47533971</td>\n",
       "      <td>21_47533971_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1056650;CLNDISDB=MONDO:MONDO:0024530,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.921060</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>21</td>\n",
       "      <td>47533980</td>\n",
       "      <td>21_47533980_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848951;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.891760</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>21</td>\n",
       "      <td>47535787</td>\n",
       "      <td>21_47535787_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=929374;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.918320</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>21</td>\n",
       "      <td>47535804</td>\n",
       "      <td>21_47535804_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848954;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.895670</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>21</td>\n",
       "      <td>47535831</td>\n",
       "      <td>21_47535831_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=860695;CLNDISDB=MedGen:CN517202;CLNDN...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.966470</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>21</td>\n",
       "      <td>47535832</td>\n",
       "      <td>21_47535832_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=939165;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.969500</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>21</td>\n",
       "      <td>47535959</td>\n",
       "      <td>21_47535959_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=848955;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.958040</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>21</td>\n",
       "      <td>47536291</td>\n",
       "      <td>21_47536291_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=792016;CLNDISDB=MONDO:MONDO:0024530,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>158810</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.883590</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>21</td>\n",
       "      <td>47536591</td>\n",
       "      <td>21_47536591_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1064926;CLNDISDB=MONDO:MONDO:0024530,...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.533000</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>K/N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>21</td>\n",
       "      <td>47537806</td>\n",
       "      <td>21_47537806_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=961546;CLNDISDB=MONDO:MONDO:0100225,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.965130</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/W</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>21</td>\n",
       "      <td>47538546</td>\n",
       "      <td>21_47538546_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1001188;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.859860</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>G/R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>21</td>\n",
       "      <td>47545423</td>\n",
       "      <td>21_47545423_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1056654;CLNDISDB=MONDO:MONDO:0024530,...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>von Willebrand factor, type A|von Willebrand f...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.896660</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>D/H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>21</td>\n",
       "      <td>47552281</td>\n",
       "      <td>21_47552281_G_A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=621016;CLNDISDB=MONDO:MONDO:0009681,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>254090</td>\n",
       "      <td>.</td>\n",
       "      <td>von Willebrand factor, type A|von Willebrand f...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.621800</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>E/K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>21</td>\n",
       "      <td>47552297</td>\n",
       "      <td>21_47552297_T_C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=608974;CLNDISDB=MONDO:MONDO:0009714,M...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>COL6A2</td>\n",
       "      <td>...</td>\n",
       "      <td>255600</td>\n",
       "      <td>.</td>\n",
       "      <td>von Willebrand factor, type A|von Willebrand f...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.482600</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>M/T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>21</td>\n",
       "      <td>47611103</td>\n",
       "      <td>21_47611103_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=822316;CLNDISDB=MONDO:MONDO:0030009,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618840</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, C-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.153260</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>T/K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>21</td>\n",
       "      <td>47614506</td>\n",
       "      <td>21_47614506_C_A</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590593;CLNDISDB=MONDO:MONDO:0014673,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>616509</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, C-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.390680</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>W/C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>21</td>\n",
       "      <td>47626603</td>\n",
       "      <td>21_47626603_T_C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=822315;CLNDISDB=MONDO:MONDO:0030009,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618840</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, C-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.218220</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>N/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>21</td>\n",
       "      <td>47630624</td>\n",
       "      <td>21_47630624_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590594;CLNDISDB=MONDO:MONDO:0032649,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618275</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, C-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.595250</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>F/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>21</td>\n",
       "      <td>47633736</td>\n",
       "      <td>21_47633736_A_C</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>AF_EXAC=0.00003;ALLELEID=590592;CLNDISDB=MONDO...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>616509</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.341080</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>I/S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>21</td>\n",
       "      <td>47636307</td>\n",
       "      <td>21_47636307_C_G</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=822317;CLNDISDB=MONDO:MONDO:0030009,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618840</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, N-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.861450</td>\n",
       "      <td>0</td>\n",
       "      <td>D</td>\n",
       "      <td>R/P</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>21</td>\n",
       "      <td>47636343</td>\n",
       "      <td>21_47636343_A_G</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590595;CLNDISDB=MONDO:MONDO:0032649,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618275</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, N-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.347000</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>L/P</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>21</td>\n",
       "      <td>47639412</td>\n",
       "      <td>21_47639412_T_A</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590598;CLNDISDB=MONDO:MONDO:0030009,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618840</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, N-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.254560</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>N/Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>21</td>\n",
       "      <td>47647481</td>\n",
       "      <td>21_47647481_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=590596;CLNDISDB=MONDO:MONDO:0032649,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618275</td>\n",
       "      <td>.</td>\n",
       "      <td>Squalene cyclase, N-terminal;Squalene cyclase,...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.074721</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>L/V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>21</td>\n",
       "      <td>47648493</td>\n",
       "      <td>21_47648493_C_T</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=822319;CLNDISDB=MONDO:MONDO:0030009,M...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>LSS</td>\n",
       "      <td>...</td>\n",
       "      <td>618840</td>\n",
       "      <td>.</td>\n",
       "      <td>.;.;.;.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.109840</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>G/D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>21</td>\n",
       "      <td>47809346</td>\n",
       "      <td>21_47809346_G_C</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=39498;CLNDISDB=MONDO:MONDO:0008872,Me...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>PCNT</td>\n",
       "      <td>...</td>\n",
       "      <td>210720</td>\n",
       "      <td>ORPHA2637</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.076200</td>\n",
       "      <td>0</td>\n",
       "      <td>T</td>\n",
       "      <td>Q/H</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>106 rows × 656 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    #CHROM       POS               ID REF ALT QUAL FILTER  \\\n",
       "0       21  27012134  21_27012134_A_G   A   G    .      .   \n",
       "1       21  27066149  21_27066149_G_A   G   A    .      .   \n",
       "2       21  27071098  21_27071098_G_C   G   C    .      .   \n",
       "3       21  27264120  21_27264120_C_T   C   T    .      .   \n",
       "4       21  28213391  21_28213391_G_T   G   T    .      .   \n",
       "5       21  33038797  21_33038797_T_C   T   C    .      .   \n",
       "6       21  33039674  21_33039674_G_A   G   A    .      .   \n",
       "7       21  33040802  21_33040802_G_A   G   A    .      .   \n",
       "8       21  33040803  21_33040803_A_C   A   C    .      .   \n",
       "9       21  33040822  21_33040822_T_G   T   G    .      .   \n",
       "10      21  33040826  21_33040826_G_A   G   A    .      .   \n",
       "11      21  33040839  21_33040839_C_T   C   T    .      .   \n",
       "12      21  34775850  21_34775850_A_C   A   C    .      .   \n",
       "13      21  34775850  21_34775850_A_G   A   G    .      .   \n",
       "14      21  34889711  21_34889711_A_C   A   C    .      .   \n",
       "15      21  34945648  21_34945648_G_A   G   A    .      .   \n",
       "16      21  34950680  21_34950680_G_A   G   A    .      .   \n",
       "17      21  34950680  21_34950680_G_A   G   A    .      .   \n",
       "18      21  34950680  21_34950680_G_A   G   A    .      .   \n",
       "19      21  34950680  21_34950680_G_A   G   A    .      .   \n",
       "20      21  34955949  21_34955949_T_C   T   C    .      .   \n",
       "21      21  34957011  21_34957011_G_A   G   A    .      .   \n",
       "22      21  34957050  21_34957050_G_A   G   A    .      .   \n",
       "23      21  34958396  21_34958396_A_G   A   G    .      .   \n",
       "24      21  35254586  21_35254586_C_T   C   T    .      .   \n",
       "25      21  35821932  21_35821932_T_A   T   A    .      .   \n",
       "26      21  35821932  21_35821932_T_A   T   A    .      .   \n",
       "27      21  36164605  21_36164605_A_C   A   C    .      .   \n",
       "28      21  36164619  21_36164619_A_C   A   C    .      .   \n",
       "29      21  36231791  21_36231791_T_A   T   A    .      .   \n",
       "30      21  36231798  21_36231798_T_C   T   C    .      .   \n",
       "31      21  36252856  21_36252856_C_A   C   A    .      .   \n",
       "32      21  36252959  21_36252959_C_T   C   T    .      .   \n",
       "33      21  36259175  21_36259175_A_T   A   T    .      .   \n",
       "34      21  36259176  21_36259176_G_T   G   T    .      .   \n",
       "35      21  36259177  21_36259177_T_G   T   G    .      .   \n",
       "36      21  36259394  21_36259394_C_T   C   T    .      .   \n",
       "37      21  36259394  21_36259394_C_T   C   T    .      .   \n",
       "38      21  37833803  21_37833803_C_T   C   T    .      .   \n",
       "39      21  38126584  21_38126584_T_G   T   G    .      .   \n",
       "40      21  38853123  21_38853123_G_A   G   A    .      .   \n",
       "41      21  38862726  21_38862726_T_G   T   G    .      .   \n",
       "42      21  38865339  21_38865339_T_A   T   A    .      .   \n",
       "43      21  38865409  21_38865409_G_T   G   T    .      .   \n",
       "44      21  38877608  21_38877608_G_A   G   A    .      .   \n",
       "45      21  40570769  21_40570769_T_A   T   A    .      .   \n",
       "46      21  40665912  21_40665912_C_T   C   T    .      .   \n",
       "47      21  41455850  21_41455850_C_T   C   T    .      .   \n",
       "48      21  41561159  21_41561159_G_A   G   A    .      .   \n",
       "49      21  43231165  21_43231165_C_T   C   T    .      .   \n",
       "50      21  43800309  21_43800309_G_T   G   T    .      .   \n",
       "51      21  43803153  21_43803153_G_T   G   T    .      .   \n",
       "52      21  43809053  21_43809053_C_T   C   T    .      .   \n",
       "53      21  44282449  21_44282449_C_T   C   T    .      .   \n",
       "54      21  44483076  21_44483076_A_G   A   G    .      .   \n",
       "55      21  44483076  21_44483076_A_G   A   G    .      .   \n",
       "56      21  44483148  21_44483148_G_T   G   T    .      .   \n",
       "57      21  44483148  21_44483148_G_T   G   T    .      .   \n",
       "58      21  44485800  21_44485800_C_T   C   T    .      .   \n",
       "59      21  44485800  21_44485800_C_T   C   T    .      .   \n",
       "60      21  44486353  21_44486353_C_T   C   T    .      .   \n",
       "61      21  44486353  21_44486353_C_T   C   T    .      .   \n",
       "62      21  44488651  21_44488651_A_G   A   G    .      .   \n",
       "63      21  44488651  21_44488651_A_G   A   G    .      .   \n",
       "64      21  45050277  21_45050277_G_A   G   A    .      .   \n",
       "65      21  45161630  21_45161630_T_A   T   A    .      .   \n",
       "66      21  45173500  21_45173500_G_A   G   A    .      .   \n",
       "67      21  45509730  21_45509730_C_T   C   T    .      .   \n",
       "68      21  45705914  21_45705914_C_T   C   T    .      .   \n",
       "69      21  45705933  21_45705933_G_A   G   A    .      .   \n",
       "70      21  45752925  21_45752925_C_T   C   T    .      .   \n",
       "71      21  46595995  21_46595995_A_G   A   G    .      .   \n",
       "72      21  46604933  21_46604933_A_G   A   G    .      .   \n",
       "73      21  46624592  21_46624592_G_A   G   A    .      .   \n",
       "74      21  46642051  21_46642051_C_T   C   T    .      .   \n",
       "75      21  47408999  21_47408999_G_A   G   A    .      .   \n",
       "76      21  47409523  21_47409523_G_A   G   A    .      .   \n",
       "77      21  47409540  21_47409540_G_C   G   C    .      .   \n",
       "78      21  47409541  21_47409541_G_A   G   A    .      .   \n",
       "79      21  47409541  21_47409541_G_T   G   T    .      .   \n",
       "80      21  47417635  21_47417635_C_T   C   T    .      .   \n",
       "81      21  47533971  21_47533971_G_T   G   T    .      .   \n",
       "82      21  47533980  21_47533980_G_A   G   A    .      .   \n",
       "83      21  47535787  21_47535787_G_T   G   T    .      .   \n",
       "84      21  47535804  21_47535804_G_A   G   A    .      .   \n",
       "85      21  47535831  21_47535831_G_C   G   C    .      .   \n",
       "86      21  47535832  21_47535832_G_T   G   T    .      .   \n",
       "87      21  47535959  21_47535959_G_C   G   C    .      .   \n",
       "88      21  47536291  21_47536291_G_T   G   T    .      .   \n",
       "89      21  47536591  21_47536591_G_C   G   C    .      .   \n",
       "90      21  47537806  21_47537806_G_T   G   T    .      .   \n",
       "91      21  47538546  21_47538546_G_A   G   A    .      .   \n",
       "92      21  47545423  21_47545423_G_C   G   C    .      .   \n",
       "93      21  47552281  21_47552281_G_A   G   A    .      .   \n",
       "94      21  47552297  21_47552297_T_C   T   C    .      .   \n",
       "95      21  47611103  21_47611103_G_T   G   T    .      .   \n",
       "96      21  47614506  21_47614506_C_A   C   A    .      .   \n",
       "97      21  47626603  21_47626603_T_C   T   C    .      .   \n",
       "98      21  47630624  21_47630624_A_G   A   G    .      .   \n",
       "99      21  47633736  21_47633736_A_C   A   C    .      .   \n",
       "100     21  47636307  21_47636307_C_G   C   G    .      .   \n",
       "101     21  47636343  21_47636343_A_G   A   G    .      .   \n",
       "102     21  47639412  21_47639412_T_A   T   A    .      .   \n",
       "103     21  47647481  21_47647481_G_C   G   C    .      .   \n",
       "104     21  47648493  21_47648493_C_T   C   T    .      .   \n",
       "105     21  47809346  21_47809346_G_C   G   C    .      .   \n",
       "\n",
       "                                                  INFO  \\\n",
       "0    ALLELEID=818128;CLNDISDB=MONDO:MONDO:0032938,M...   \n",
       "1    ALLELEID=818131;CLNDISDB=MONDO:MONDO:0032938,M...   \n",
       "2    ALLELEID=818129;CLNDISDB=MONDO:MONDO:0032938,M...   \n",
       "3    ALLELEID=626283;CLNDISDB=MONDO:MONDO:0005620,M...   \n",
       "4    ALLELEID=918095;CLNDISDB=MONDO:MONDO:0005387,M...   \n",
       "5    ALLELEID=861431;CLNDISDB=MONDO:MONDO:0007103,M...   \n",
       "6    ALLELEID=920983;CLNDISDB=MedGen:CN517202;CLNDN...   \n",
       "7    ALLELEID=861434;CLNDISDB=MONDO:MONDO:0007103,M...   \n",
       "8    ALLELEID=961894;CLNDISDB=MONDO:MONDO:0007103,M...   \n",
       "9    ALLELEID=1052613;CLNDISDB=MONDO:MONDO:0007103,...   \n",
       "10   ALLELEID=861435;CLNDISDB=MONDO:MONDO:0007103,M...   \n",
       "11   ALLELEID=1152912;CLNDISDB=MedGen:CN517202;CLND...   \n",
       "12   ALLELEID=1056638;CLNDISDB=MONDO:MONDO:0013953,...   \n",
       "13   ALLELEID=975708;CLNDISDB=MONDO:MONDO:0013953,M...   \n",
       "14   ALLELEID=904905;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "15   ALLELEID=815961;CLNDISDB=MONDO:MONDO:0014936,M...   \n",
       "16   ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "17   ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "18   ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "19   ALLELEID=1162253;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "20   ALLELEID=999978;CLNDISDB=.|MONDO:MONDO:0016817...   \n",
       "21   ALLELEID=999979;CLNDISDB=.|MONDO:MONDO:0016817...   \n",
       "22   ALLELEID=1162254;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "23   ALLELEID=1162256;CLNDISDB=MONDO:MONDO:0016817,...   \n",
       "24   ALLELEID=904906;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "25   ALLELEID=1056640;CLNDISDB=MONDO:MONDO:0002442,...   \n",
       "26   ALLELEID=1056640;CLNDISDB=MONDO:MONDO:0002442,...   \n",
       "27   ALLELEID=976801;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "28   ALLELEID=976803;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "29   ALLELEID=615635;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "30   ALLELEID=976804;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "31   ALLELEID=857423;CLNDISDB=MONDO:MONDO:0100083,M...   \n",
       "32   ALLELEID=976806;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "33   ALLELEID=430422;CLNDISDB=MONDO:MONDO:0100083,M...   \n",
       "34   ALLELEID=653831;CLNDISDB=MONDO:MONDO:0011071,M...   \n",
       "35   ALLELEID=552387;CLNDISDB=MONDO:MONDO:0100083,M...   \n",
       "36   ALLELEID=976809;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "37   ALLELEID=976809;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "38   ALLELEID=590337;CLNDISDB=MONDO:MONDO:0013537,M...   \n",
       "39   AF_EXAC=0.00003;ALLELEID=792007;CLNDISDB=MONDO...   \n",
       "40   ALLELEID=648994;CLNDISDB=MONDO:MONDO:0013578,M...   \n",
       "41   ALLELEID=1151744;CLNDISDB=Human_Phenotype_Onto...   \n",
       "42   ALLELEID=615979;CLNDISDB=MONDO:MONDO:0013578,M...   \n",
       "43   ALLELEID=920984;CLNDISDB=MedGen:CN517202;CLNDN...   \n",
       "44   ALLELEID=951237;CLNDISDB=MONDO:MONDO:0013578,M...   \n",
       "45   ALLELEID=970556;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "46   ALLELEID=918097;CLNDISDB=MONDO:MONDO:0005387,M...   \n",
       "47   ALLELEID=1087103;CLNDISDB=.;CLNDN=See_cases;CL...   \n",
       "48   ALLELEID=590759;CLNDISDB=MONDO:MONDO:0018309,M...   \n",
       "49   ALLELEID=860689;CLNDISDB=MedGen:CN517202;CLNDN...   \n",
       "50   ALLELEID=1053403;CLNDISDB=Human_Phenotype_Onto...   \n",
       "51   ALLELEID=962918;CLNDISDB=MONDO:MONDO:0010987,M...   \n",
       "52   ALLELEID=1053405;CLNDISDB=Human_Phenotype_Onto...   \n",
       "53   ALLELEID=610999;CLNDISDB=MONDO:MONDO:0032691,M...   \n",
       "54   ALLELEID=966848;CLNDISDB=MONDO:MONDO:0009352,M...   \n",
       "55   ALLELEID=966848;CLNDISDB=MONDO:MONDO:0009352,M...   \n",
       "56   ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...   \n",
       "57   ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...   \n",
       "58   AF_EXAC=0.00004;ALLELEID=210437;CLNDISDB=Human...   \n",
       "59   AF_EXAC=0.00004;ALLELEID=210437;CLNDISDB=Human...   \n",
       "60   ALLELEID=848818;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "61   ALLELEID=848818;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "62   ALLELEID=471673;CLNDISDB=MONDO:MONDO:0009352,M...   \n",
       "63   ALLELEID=471673;CLNDISDB=MONDO:MONDO:0009352,M...   \n",
       "64   ALLELEID=1035760;CLNDISDB=MONDO:MONDO:0030985,...   \n",
       "65   ALLELEID=800762;CLNDISDB=MONDO:MONDO:0032792,M...   \n",
       "66   AF_EXAC=0.00001;ALLELEID=624077;CLNDISDB=MONDO...   \n",
       "67   ALLELEID=984452;CLNDISDB=Human_Phenotype_Ontol...   \n",
       "68   ALLELEID=961986;CLNDISDB=MONDO:MONDO:0009411,M...   \n",
       "69   ALLELEID=980969;CLNDISDB=MONDO:MONDO:0009411,M...   \n",
       "70   ALLELEID=800625;CLNDISDB=.;CLNDN=Cone-rod_dege...   \n",
       "71   ALLELEID=858619;CLNDISDB=MONDO:MONDO:0030025,M...   \n",
       "72   ALLELEID=858618;CLNDISDB=MONDO:MONDO:0030025,M...   \n",
       "73   ALLELEID=858620;CLNDISDB=MONDO:MONDO:0030025,M...   \n",
       "74   ALLELEID=858621;CLNDISDB=MONDO:MONDO:0030025,M...   \n",
       "75   ALLELEID=860694;CLNDISDB=MedGen:CN517202;CLNDN...   \n",
       "76   ALLELEID=848918;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "77   ALLELEID=1064923;CLNDISDB=MONDO:MONDO:0024530,...   \n",
       "78   ALLELEID=649076;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "79   ALLELEID=929366;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "80   ALLELEID=608972;CLNDISDB=MONDO:MONDO:0009681,M...   \n",
       "81   ALLELEID=1056650;CLNDISDB=MONDO:MONDO:0024530,...   \n",
       "82   ALLELEID=848951;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "83   ALLELEID=929374;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "84   ALLELEID=848954;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "85   ALLELEID=860695;CLNDISDB=MedGen:CN517202;CLNDN...   \n",
       "86   ALLELEID=939165;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "87   ALLELEID=848955;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "88   ALLELEID=792016;CLNDISDB=MONDO:MONDO:0024530,M...   \n",
       "89   ALLELEID=1064926;CLNDISDB=MONDO:MONDO:0024530,...   \n",
       "90   ALLELEID=961546;CLNDISDB=MONDO:MONDO:0100225,M...   \n",
       "91   ALLELEID=1001188;CLNDISDB=MedGen:CN517202;CLND...   \n",
       "92   ALLELEID=1056654;CLNDISDB=MONDO:MONDO:0024530,...   \n",
       "93   ALLELEID=621016;CLNDISDB=MONDO:MONDO:0009681,M...   \n",
       "94   ALLELEID=608974;CLNDISDB=MONDO:MONDO:0009714,M...   \n",
       "95   ALLELEID=822316;CLNDISDB=MONDO:MONDO:0030009,M...   \n",
       "96   ALLELEID=590593;CLNDISDB=MONDO:MONDO:0014673,M...   \n",
       "97   ALLELEID=822315;CLNDISDB=MONDO:MONDO:0030009,M...   \n",
       "98   ALLELEID=590594;CLNDISDB=MONDO:MONDO:0032649,M...   \n",
       "99   AF_EXAC=0.00003;ALLELEID=590592;CLNDISDB=MONDO...   \n",
       "100  ALLELEID=822317;CLNDISDB=MONDO:MONDO:0030009,M...   \n",
       "101  ALLELEID=590595;CLNDISDB=MONDO:MONDO:0032649,M...   \n",
       "102  ALLELEID=590598;CLNDISDB=MONDO:MONDO:0030009,M...   \n",
       "103  ALLELEID=590596;CLNDISDB=MONDO:MONDO:0032649,M...   \n",
       "104  ALLELEID=822319;CLNDISDB=MONDO:MONDO:0030009,M...   \n",
       "105  ALLELEID=39498;CLNDISDB=MONDO:MONDO:0008872,Me...   \n",
       "\n",
       "                           CLNSIG      Gene  ...  clinvar_OMIM_id  \\\n",
       "0                      Pathogenic      JAM2  ...           618824   \n",
       "1                      Pathogenic      JAM2  ...           618824   \n",
       "2                      Pathogenic      JAM2  ...           618824   \n",
       "3               Likely_pathogenic       APP  ...           605714   \n",
       "4               Likely_pathogenic   ADAMTS1  ...         PS311360   \n",
       "5               Likely_pathogenic      SOD1  ...           105400   \n",
       "6               Likely_pathogenic      SOD1  ...                .   \n",
       "7               Likely_pathogenic      SOD1  ...           105400   \n",
       "8               Likely_pathogenic      SOD1  ...           105400   \n",
       "9               Likely_pathogenic      SOD1  ...                .   \n",
       "10              Likely_pathogenic      SOD1  ...           105400   \n",
       "11              Likely_pathogenic      SOD1  ...                .   \n",
       "12              Likely_pathogenic    IFNGR2  ...                .   \n",
       "13                     Pathogenic    IFNGR2  ...           614889   \n",
       "14              Likely_pathogenic      GART  ...           189960   \n",
       "15                     Pathogenic       SON  ...           617140   \n",
       "16                     Pathogenic    DONSON  ...                .   \n",
       "17                     Pathogenic    DONSON  ...                .   \n",
       "18                     Pathogenic    DONSON  ...                .   \n",
       "19                     Pathogenic    DONSON  ...                .   \n",
       "20                     Pathogenic    DONSON  ...                .   \n",
       "21                     Pathogenic    DONSON  ...                .   \n",
       "22                     Pathogenic    DONSON  ...                .   \n",
       "23                     Pathogenic    DONSON  ...                .   \n",
       "24              Likely_pathogenic     ITSN1  ...           189960   \n",
       "25              Likely_pathogenic     KCNE1  ...                .   \n",
       "26              Likely_pathogenic     KCNE1  ...                .   \n",
       "27              Likely_pathogenic     RUNX1  ...                .   \n",
       "28              Likely_pathogenic     RUNX1  ...                .   \n",
       "29              Likely_pathogenic     RUNX1  ...           601399   \n",
       "30              Likely_pathogenic     RUNX1  ...                .   \n",
       "31              Likely_pathogenic     RUNX1  ...           601399   \n",
       "32              Likely_pathogenic     RUNX1  ...                .   \n",
       "33              Likely_pathogenic     RUNX1  ...           601399   \n",
       "34              Likely_pathogenic     RUNX1  ...           601399   \n",
       "35              Likely_pathogenic     RUNX1  ...           601399   \n",
       "36                     Pathogenic     RUNX1  ...                .   \n",
       "37                     Pathogenic     RUNX1  ...                .   \n",
       "38              Likely_pathogenic    CLDN14  ...           614035   \n",
       "39              Likely_pathogenic      HLCS  ...           253270   \n",
       "40              Likely_pathogenic    DYRK1A  ...           614104   \n",
       "41              Likely_pathogenic    DYRK1A  ...                .   \n",
       "42              Likely_pathogenic    DYRK1A  ...           614104   \n",
       "43                     Pathogenic    DYRK1A  ...                .   \n",
       "44              Likely_pathogenic    DYRK1A  ...           614104   \n",
       "45              Likely_pathogenic     BRWD1  ...                .   \n",
       "46              Likely_pathogenic     BRWD1  ...         PS311360   \n",
       "47              Likely_pathogenic     DSCAM  ...                .   \n",
       "48              Likely_pathogenic     DSCAM  ...         PS142623   \n",
       "49              Likely_pathogenic    PRDM15  ...                .   \n",
       "50              Likely_pathogenic   TMPRSS3  ...                .   \n",
       "51              Likely_pathogenic   TMPRSS3  ...           601072   \n",
       "52              Likely_pathogenic   TMPRSS3  ...                .   \n",
       "53              Likely_pathogenic      WDR4  ...           618347   \n",
       "54                     Pathogenic       CBS  ...                .   \n",
       "55                     Pathogenic       CBS  ...           236200   \n",
       "56              Likely_pathogenic       CBS  ...                .   \n",
       "57              Likely_pathogenic       CBS  ...                .   \n",
       "58   Pathogenic/Likely_pathogenic       CBS  ...                .   \n",
       "59   Pathogenic/Likely_pathogenic       CBS  ...           236200   \n",
       "60                     Pathogenic       CBS  ...                .   \n",
       "61                     Pathogenic       CBS  ...           236200   \n",
       "62                     Pathogenic       CBS  ...                .   \n",
       "63                     Pathogenic       CBS  ...           236200   \n",
       "64                     Pathogenic    HSF2BP  ...                .   \n",
       "65              Likely_pathogenic      PDXK  ...           618511   \n",
       "66                     Pathogenic      PDXK  ...           618511   \n",
       "67              Likely_pathogenic  TRAPPC10  ...                .   \n",
       "68              Likely_pathogenic      AIRE  ...           240300   \n",
       "69              Likely_pathogenic      AIRE  ...           240300   \n",
       "70              Likely_pathogenic   CFAP410  ...                .   \n",
       "71                     Pathogenic    ADARB1  ...           618862   \n",
       "72                     Pathogenic    ADARB1  ...           618862   \n",
       "73                     Pathogenic    ADARB1  ...           618862   \n",
       "74                     Pathogenic    ADARB1  ...           618862   \n",
       "75                     Pathogenic    COL6A1  ...                .   \n",
       "76                     Pathogenic    COL6A1  ...           158810   \n",
       "77                     Pathogenic    COL6A1  ...                .   \n",
       "78                     Pathogenic    COL6A1  ...           158810   \n",
       "79              Likely_pathogenic    COL6A1  ...           158810   \n",
       "80              Likely_pathogenic    COL6A1  ...           254090   \n",
       "81              Likely_pathogenic    COL6A2  ...                .   \n",
       "82                     Pathogenic    COL6A2  ...           158810   \n",
       "83              Likely_pathogenic    COL6A2  ...           158810   \n",
       "84              Likely_pathogenic    COL6A2  ...           158810   \n",
       "85                     Pathogenic    COL6A2  ...                .   \n",
       "86              Likely_pathogenic    COL6A2  ...           158810   \n",
       "87                     Pathogenic    COL6A2  ...           158810   \n",
       "88                     Pathogenic    COL6A2  ...           158810   \n",
       "89                     Pathogenic    COL6A2  ...                .   \n",
       "90              Likely_pathogenic    COL6A2  ...                .   \n",
       "91              Likely_pathogenic    COL6A2  ...                .   \n",
       "92              Likely_pathogenic    COL6A2  ...                .   \n",
       "93              Likely_pathogenic    COL6A2  ...           254090   \n",
       "94              Likely_pathogenic    COL6A2  ...           255600   \n",
       "95                     Pathogenic       LSS  ...           618840   \n",
       "96                     Pathogenic       LSS  ...           616509   \n",
       "97                     Pathogenic       LSS  ...           618840   \n",
       "98                     Pathogenic       LSS  ...           618275   \n",
       "99                     Pathogenic       LSS  ...           616509   \n",
       "100                    Pathogenic       LSS  ...           618840   \n",
       "101                    Pathogenic       LSS  ...           618275   \n",
       "102                    Pathogenic       LSS  ...           618840   \n",
       "103                    Pathogenic       LSS  ...           618275   \n",
       "104                    Pathogenic       LSS  ...           618840   \n",
       "105                    Pathogenic      PCNT  ...           210720   \n",
       "\n",
       "     clinvar_Orphanet_id                                    Interpro_domain  \\\n",
       "0                      .                                              .;.;.   \n",
       "1                      .  .;Immunoglobulin V-set domain|Immunoglobulin-l...   \n",
       "2                      .  .;Immunoglobulin-like domain|Immunoglobulin su...   \n",
       "3             ORPHA85458  Amyloidogenic glycoprotein, amyloid-beta pepti...   \n",
       "4                      .  Peptidase M12B, ADAM/reprolysin|Peptidase M12B...   \n",
       "5                      .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "6                      .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "7                      .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "8                      .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "9                      .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "10                     .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "11                     .  Superoxide dismutase, copper/zinc binding doma...   \n",
       "12                     .                                                .;.   \n",
       "13                     .                                                .;.   \n",
       "14             ORPHA1199  PurM-like, C-terminal domain;PurM-like, C-term...   \n",
       "15           ORPHA500150  G-patch domain|G-patch domain|G-patch domain;G...   \n",
       "16                     .                                              .;.;.   \n",
       "17                     .                                              .;.;.   \n",
       "18                     .                                                  .   \n",
       "19                     .                                                  .   \n",
       "20                     .                                            .;.;.;.   \n",
       "21                     .                                          .;.;.;.;.   \n",
       "22                     .                                            .;.;.;.   \n",
       "23                     .                                          .;.;.;.;.   \n",
       "24             ORPHA1199                       Pleckstrin homology domain;.   \n",
       "25                     .                                            .;.;.;.   \n",
       "26                     .                                    .;.;.;.;.;.;.;.   \n",
       "27                     .  Runx, C-terminal  domain;.;.;Runx, C-terminal ...   \n",
       "28                     .  Runx, C-terminal  domain;.;.;Runx, C-terminal ...   \n",
       "29                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "30                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "31                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "32                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "33                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "34                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "35                     .  Runt domain|Runt domain;.;.;Runt domain|Runt d...   \n",
       "36                     .                                          .;.;.;.;.   \n",
       "37                     .                                              .;.;.   \n",
       "38                     .  Claudin, conserved site;Claudin, conserved sit...   \n",
       "39            ORPHA79242  Biotin protein ligase, C-terminal;Biotin prote...   \n",
       "40           ORPHA464306  Protein kinase domain|Protein kinase, ATP bind...   \n",
       "41                     .  Protein kinase domain|Protein kinase domain|Pr...   \n",
       "42           ORPHA464306  Protein kinase domain|Protein kinase domain|Pr...   \n",
       "43                     .  Protein kinase domain|Protein kinase domain|Pr...   \n",
       "44           ORPHA464306  Protein kinase domain|Protein kinase domain|Pr...   \n",
       "45                     .                                              .;.;.   \n",
       "46                     .                  WD40-repeat-containing domain;.;.   \n",
       "47                     .  Fibronectin type III|Fibronectin type III|Fibr...   \n",
       "48              ORPHA388  Immunoglobulin-like domain;Immunoglobulin-like...   \n",
       "49                     .  .;.;Zinc finger C2H2-type|Zinc finger C2H2-typ...   \n",
       "50                     .  .;Serine proteases, trypsin domain|Serine prot...   \n",
       "51                     .  .;Serine proteases, trypsin domain|Serine prot...   \n",
       "52                     .  .;Low-density lipoprotein (LDL) receptor class...   \n",
       "53                     .  WD40-repeat-containing domain;WD40-repeat-cont...   \n",
       "54                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "55              ORPHA394  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "56                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "57                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "58                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "59              ORPHA394  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "60                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "61              ORPHA394  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "62                     .  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "63              ORPHA394  Pyridoxal-phosphate dependent enzyme;Pyridoxal...   \n",
       "64                     .                                                .;.   \n",
       "65                     .                                              .;.;.   \n",
       "66                     .  .;Pyridoxamine kinase/Phosphomethylpyrimidine ...   \n",
       "67                     .                                                  .   \n",
       "68             ORPHA3453                              HSR domain|HSR domain   \n",
       "69             ORPHA3453                              HSR domain|HSR domain   \n",
       "70                     .                                              .;.;.   \n",
       "71                     .  Double-stranded RNA-binding domain|Double-stra...   \n",
       "72                     .  Adenosine deaminase/editase|Adenosine deaminas...   \n",
       "73                     .  Adenosine deaminase/editase|Adenosine deaminas...   \n",
       "74                     .  Adenosine deaminase/editase|Adenosine deaminas...   \n",
       "75                     .                                                .;.   \n",
       "76                     .                                                .;.   \n",
       "77                     .                                                .;.   \n",
       "78                     .                                                .;.   \n",
       "79                     .                                                .;.   \n",
       "80                     .                                                .;.   \n",
       "81                     .                                            .;.;.;.   \n",
       "82                     .                                            .;.;.;.   \n",
       "83                     .                                            .;.;.;.   \n",
       "84                     .                                            .;.;.;.   \n",
       "85                     .                                            .;.;.;.   \n",
       "86                     .                                            .;.;.;.   \n",
       "87                     .                                            .;.;.;.   \n",
       "88                     .                                            .;.;.;.   \n",
       "89                     .                                            .;.;.;.   \n",
       "90                     .                                            .;.;.;.   \n",
       "91                     .                                            .;.;.;.   \n",
       "92                     .  von Willebrand factor, type A|von Willebrand f...   \n",
       "93                     .  von Willebrand factor, type A|von Willebrand f...   \n",
       "94                     .  von Willebrand factor, type A|von Willebrand f...   \n",
       "95                     .  Squalene cyclase, C-terminal;Squalene cyclase,...   \n",
       "96                     .  Squalene cyclase, C-terminal;Squalene cyclase,...   \n",
       "97                     .  Squalene cyclase, C-terminal;Squalene cyclase,...   \n",
       "98                     .  Squalene cyclase, C-terminal;Squalene cyclase,...   \n",
       "99                     .                                            .;.;.;.   \n",
       "100                    .  Squalene cyclase, N-terminal;Squalene cyclase,...   \n",
       "101                    .  Squalene cyclase, N-terminal;Squalene cyclase,...   \n",
       "102                    .  Squalene cyclase, N-terminal;Squalene cyclase,...   \n",
       "103                    .  Squalene cyclase, N-terminal;Squalene cyclase,...   \n",
       "104                    .                                            .;.;.;.   \n",
       "105            ORPHA2637                                                  .   \n",
       "\n",
       "    GTEx_V8_gene GTEx_V8_tissue Geuvadis_eQTL_target_gene MISTIC_score  \\\n",
       "0              .              .                         .     0.242621   \n",
       "1              .              .                         .     0.641090   \n",
       "2              .              .                         .     0.933320   \n",
       "3              .              .                         .     0.799610   \n",
       "4              .              .                         .     0.152990   \n",
       "5              .              .                         .     0.888600   \n",
       "6              .              .                         .     0.913210   \n",
       "7              .              .                         .     0.928640   \n",
       "8              .              .                         .     0.939960   \n",
       "9              .              .                         .     0.738720   \n",
       "10             .              .                         .     0.926230   \n",
       "11             .              .                         .     0.843900   \n",
       "12             .              .                         .     0.165504   \n",
       "13             .              .                         .     0.154573   \n",
       "14             .              .                         .     0.390680   \n",
       "15             .              .                         .     0.713870   \n",
       "16             .              .                         .     0.233880   \n",
       "17             .              .                         .     0.233880   \n",
       "18             .              .                         .     0.233880   \n",
       "19             .              .                         .     0.233880   \n",
       "20             .              .                         .     0.654430   \n",
       "21             .              .                         .     0.721110   \n",
       "22             .              .                         .     0.679830   \n",
       "23             .              .                         .     0.495180   \n",
       "24             .              .                         .     0.914020   \n",
       "25             .              .                         .     0.132754   \n",
       "26             .              .                         .     0.405664   \n",
       "27             .              .                         .     0.218180   \n",
       "28             .              .                         .     0.099688   \n",
       "29             .              .                         .     0.984310   \n",
       "30             .              .                         .     0.984580   \n",
       "31             .              .                         .     0.976880   \n",
       "32             .              .                         .     0.968070   \n",
       "33             .              .                         .     0.967160   \n",
       "34             .              .                         .     0.909230   \n",
       "35             .              .                         .     0.953190   \n",
       "36             .              .                         .     0.906094   \n",
       "37             .              .                         .     0.906094   \n",
       "38             .              .                         .     0.903510   \n",
       "39             .              .                         .     0.751810   \n",
       "40             .              .                         .     0.971090   \n",
       "41             .              .                         .     0.895650   \n",
       "42             .              .                         .     0.962630   \n",
       "43             .              .                         .     0.958940   \n",
       "44             .              .                         .     0.761110   \n",
       "45             .              .                         .     0.118330   \n",
       "46             .              .                         .     0.216200   \n",
       "47             .              .                         .     0.492790   \n",
       "48             .              .                         .     0.247440   \n",
       "49             .              .                         .     0.965100   \n",
       "50             .              .                         .     0.821610   \n",
       "51             .              .                         .     0.941820   \n",
       "52             .              .                         .     0.921870   \n",
       "53             .              .                         .     0.680960   \n",
       "54             .              .                         .     0.866630   \n",
       "55             .              .                         .     0.866630   \n",
       "56             .              .                         .     0.805350   \n",
       "57             .              .                         .     0.805350   \n",
       "58             .              .                         .     0.955000   \n",
       "59             .              .                         .     0.955000   \n",
       "60             .              .                         .     0.865950   \n",
       "61             .              .                         .     0.865950   \n",
       "62             .              .                         .     0.673380   \n",
       "63             .              .                         .     0.673380   \n",
       "64             .              .                         .     0.340330   \n",
       "65             .              .                         .     0.564970   \n",
       "66             .              .                         .     0.228580   \n",
       "67             .              .                         .     0.551070   \n",
       "68             .              .                         .     0.722000   \n",
       "69             .              .                         .     0.934830   \n",
       "70             .              .                         .     0.499090   \n",
       "71             .              .                         .     0.910570   \n",
       "72             .              .                         .     0.263620   \n",
       "73             .              .                         .     0.776460   \n",
       "74             .              .                         .     0.651410   \n",
       "75             .              .                         .     0.776730   \n",
       "76             .              .                         .     0.875210   \n",
       "77             .              .                         .     0.877090   \n",
       "78             .              .                         .     0.877050   \n",
       "79             .              .                         .     0.906860   \n",
       "80             .              .                         .     0.677970   \n",
       "81             .              .                         .     0.921060   \n",
       "82             .              .                         .     0.891760   \n",
       "83             .              .                         .     0.918320   \n",
       "84             .              .                         .     0.895670   \n",
       "85             .              .                         .     0.966470   \n",
       "86             .              .                         .     0.969500   \n",
       "87             .              .                         .     0.958040   \n",
       "88             .              .                         .     0.883590   \n",
       "89             .              .                         .     0.533000   \n",
       "90             .              .                         .     0.965130   \n",
       "91             .              .                         .     0.859860   \n",
       "92             .              .                         .     0.896660   \n",
       "93             .              .                         .     0.621800   \n",
       "94             .              .                         .     0.482600   \n",
       "95             .              .                         .     0.153260   \n",
       "96             .              .                         .     0.390680   \n",
       "97             .              .                         .     0.218220   \n",
       "98             .              .                         .     0.595250   \n",
       "99             .              .                         .     0.341080   \n",
       "100            .              .                         .     0.861450   \n",
       "101            .              .                         .     0.347000   \n",
       "102            .              .                         .     0.254560   \n",
       "103            .              .                         .     0.074721   \n",
       "104            .              .                         .     0.109840   \n",
       "105            .              .                         .     0.076200   \n",
       "\n",
       "    MISTIC_sd MISTIC_pred MISTIC_comments  \n",
       "0           0           T             M/V  \n",
       "1           0           D             R/H  \n",
       "2           0           D             W/C  \n",
       "3           0           D             G/S  \n",
       "4           0           T             S/Y  \n",
       "5           0           D             S/P  \n",
       "6           0           D             G/S  \n",
       "7           0           D             D/N  \n",
       "8           0           D             D/A  \n",
       "9           0           D             N/K  \n",
       "10          0           D             E/K  \n",
       "11          0           D             T/I  \n",
       "12          0           T             M/L  \n",
       "13          0           T             M/V  \n",
       "14          0           T             L/R  \n",
       "15          0           D             G/E  \n",
       "16          0           T             P/L  \n",
       "17          0           T             R/C  \n",
       "18          0           T             P/L  \n",
       "19          0           T             R/C  \n",
       "20          0           D             Y/C  \n",
       "21          0           D             P/S  \n",
       "22          0           D             R/C  \n",
       "23          0           T             F/S  \n",
       "24          0           D             R/C  \n",
       "25          0           T             M/L  \n",
       "26          0           T             M/L  \n",
       "27          0           T             S/A  \n",
       "28          0           T             V/G  \n",
       "29          0           D             D/V  \n",
       "30          0           D             T/A  \n",
       "31          0           D             R/I  \n",
       "32          0           D             G/S  \n",
       "33          0           D             W/R  \n",
       "34          0           D             H/Q  \n",
       "35          0           D             H/P  \n",
       "36          0           D             D/N  \n",
       "37          0           D             D/N  \n",
       "38          0           D             C/Y  \n",
       "39          0           D             D/A  \n",
       "40          0           D             G/R  \n",
       "41          0           D             I/R  \n",
       "42          0           D             S/R  \n",
       "43          0           D             G/W  \n",
       "44          0           D             R/H  \n",
       "45          0           T             Q/L  \n",
       "46          0           T             R/H  \n",
       "47          0           T             G/S  \n",
       "48          0           T             A/V  \n",
       "49          0           D             C/Y  \n",
       "50          0           D             P/H  \n",
       "51          0           D             H/Q  \n",
       "52          0           D             D/N  \n",
       "53          0           D             R/Q  \n",
       "54          0           D             V/A  \n",
       "55          0           D             V/A  \n",
       "56          0           D             P/Q  \n",
       "57          0           D             P/Q  \n",
       "58          0           D             G/R  \n",
       "59          0           D             G/R  \n",
       "60          0           D             G/R  \n",
       "61          0           D             G/R  \n",
       "62          0           D             I/T  \n",
       "63          0           D             I/T  \n",
       "64          0           T             S/L  \n",
       "65          0           D             N/K  \n",
       "66          0           T             R/Q  \n",
       "67          0           D             P/L  \n",
       "68          0           D             R/W  \n",
       "69          0           D             R/H  \n",
       "70          0           T             D/N  \n",
       "71          0           D             K/E  \n",
       "72          0           T             S/G  \n",
       "73          0           D             R/Q  \n",
       "74          0           D             A/V  \n",
       "75          0           D             G/E  \n",
       "76          0           D             G/E  \n",
       "77          0           D             G/R  \n",
       "78          0           D             G/E  \n",
       "79          0           D             G/V  \n",
       "80          0           D             P/S  \n",
       "81          0           D             G/V  \n",
       "82          0           D             G/E  \n",
       "83          0           D             G/V  \n",
       "84          0           D             G/S  \n",
       "85          0           D             G/R  \n",
       "86          0           D             G/V  \n",
       "87          0           D             G/R  \n",
       "88          0           D             G/C  \n",
       "89          0           D             K/N  \n",
       "90          0           D             G/W  \n",
       "91          0           D             G/R  \n",
       "92          0           D             D/H  \n",
       "93          0           D             E/K  \n",
       "94          0           T             M/T  \n",
       "95          0           T             T/K  \n",
       "96          0           T             W/C  \n",
       "97          0           T             N/S  \n",
       "98          0           D             F/S  \n",
       "99          0           T             I/S  \n",
       "100         0           D             R/P  \n",
       "101         0           T             L/P  \n",
       "102         0           T             N/Y  \n",
       "103         0           T             L/V  \n",
       "104         0           T             G/D  \n",
       "105         0           T             Q/H  \n",
       "\n",
       "[106 rows x 656 columns]"
      ]
     },
     "execution_count": 173,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mistic = pd.read_csv('/gstock/MISTIC/CAGI6/download/OUTPUT/MISTIC_chr{}.tsv'.format('21'), sep='\\t')\n",
    "pd.merge(test, mistic.rename({c:\"MISTIC_\"+c for c in list(mistic.columns[4:])}, axis=1), on=['#chr', 'pos(1-based)', 'ref', 'alt'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "id": "f76cbee7-b748-4b60-977a-503972cb3ca4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>Gene</th>\n",
       "      <th>...</th>\n",
       "      <th>aaalt</th>\n",
       "      <th>rs_dbSNP</th>\n",
       "      <th>hg19_chr</th>\n",
       "      <th>hg19_pos(1-based)</th>\n",
       "      <th>hg18_chr</th>\n",
       "      <th>hg18_pos(1-based)</th>\n",
       "      <th>aapos</th>\n",
       "      <th>genename</th>\n",
       "      <th>Ensembl_geneid</th>\n",
       "      <th>Ensembl_transcriptid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21_44483148_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>Q</td>\n",
       "      <td>.</td>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21</td>\n",
       "      <td>43356217</td>\n",
       "      <td>290;290;290;290;290</td>\n",
       "      <td>CBSL;CBSL;CBSL;CBSL;CBSL</td>\n",
       "      <td>ENSG00000274276;ENSG00000274276;ENSG0000027427...</td>\n",
       "      <td>ENST00000624406;ENST00000398168;ENST0000061802...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21_44483148_G_T</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>CBS</td>\n",
       "      <td>...</td>\n",
       "      <td>Q</td>\n",
       "      <td>rs760912339</td>\n",
       "      <td>21</td>\n",
       "      <td>44483148</td>\n",
       "      <td>21</td>\n",
       "      <td>43356217</td>\n",
       "      <td>290;290;290;290</td>\n",
       "      <td>CBS;CBS;CBS;CBS</td>\n",
       "      <td>ENSG00000160200;ENSG00000160200;ENSG0000016020...</td>\n",
       "      <td>ENST00000398158;ENST00000398165;ENST0000035962...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 25 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   #CHROM       POS               ID REF ALT QUAL FILTER  \\\n",
       "54     21  44483148  21_44483148_G_T   G   T    .      .   \n",
       "55     21  44483148  21_44483148_G_T   G   T    .      .   \n",
       "\n",
       "                                                 INFO             CLNSIG Gene  \\\n",
       "54  ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...  Likely_pathogenic  CBS   \n",
       "55  ALLELEID=1174877;CLNDISDB=MedGen:CN517202;CLND...  Likely_pathogenic  CBS   \n",
       "\n",
       "    ...  aaalt     rs_dbSNP hg19_chr hg19_pos(1-based) hg18_chr  \\\n",
       "54  ...      Q            .       21          44483148       21   \n",
       "55  ...      Q  rs760912339       21          44483148       21   \n",
       "\n",
       "   hg18_pos(1-based)                aapos                  genename  \\\n",
       "54          43356217  290;290;290;290;290  CBSL;CBSL;CBSL;CBSL;CBSL   \n",
       "55          43356217      290;290;290;290           CBS;CBS;CBS;CBS   \n",
       "\n",
       "                                       Ensembl_geneid  \\\n",
       "54  ENSG00000274276;ENSG00000274276;ENSG0000027427...   \n",
       "55  ENSG00000160200;ENSG00000160200;ENSG0000016020...   \n",
       "\n",
       "                                 Ensembl_transcriptid  \n",
       "54  ENST00000624406;ENST00000398168;ENST0000061802...  \n",
       "55  ENST00000398158;ENST00000398165;ENST0000035962...  \n",
       "\n",
       "[2 rows x 25 columns]"
      ]
     },
     "execution_count": 167,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# test = pd.merge(clinvar_2021_clean, dbnsfp_tmp, on ='ID')\n",
    "test.loc[test['ID'] == '21_44483148_G_T'][test.columns[:25]]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7848cf49-861e-4c9b-9297-bf9c15aa9be3",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# EXPORT FILES \n",
    "\n",
    "from tqdm import tqdm\n",
    "tqdm.pandas()\n",
    "import time \n",
    "\n",
    "import os, sys\n",
    "sys.path.append('/gstock/biolo_datasets/variation/benchmark/MISTIC/RADMEL_git/Complementary_tools')\n",
    "from AAIndex import AAIndex\n",
    "a = AAIndex()\n",
    "\n",
    "# Fct to retrieve AAIndex properities in AAIndex class\n",
    "def aa_index_pandas(r):\n",
    "    try : \n",
    "        s = pd.Series(a.score_missense(r['aaref'],r['aaalt']))\n",
    "        r = pd.concat([r, s])\n",
    "        return r\n",
    "    except :\n",
    "        print(r['aaref'],r['aaalt'])\n",
    "        return r\n",
    "\n",
    "log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'w')\n",
    "log.close()\n",
    "\n",
    "chrs = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', 'X', 'Y', 'M']\n",
    "for chromosome in chrs:\n",
    "    output_file_path = '/gstock/MISTIC/CAGI6/download/OUTPUT/dbnsfp.final.chr{}.csv'.format(chromosome)\n",
    "    print(\"{}{}{}\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), \" - CHROM : \", chromosome))\n",
    "    log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "    log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), \" - CHROM : \", chromosome))\n",
    "    log.close()\n",
    "    \n",
    "    if os.path.isfile(output_file_path) is False:\n",
    "        \n",
    "        # RETRIEVE DBNSFP LITE PANDAS PART CORRESPONDING TO CHR\n",
    "        dbnsfp_vcf_hail_pandas_tmp = dbnsfp_vcf_hail_pandas.loc[dbnsfp_vcf_hail_pandas['CHROM'] == chromosome]\n",
    "        \n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), ' - VEP VCF - ', dbnsfp_vcf_hail_pandas_tmp.shape))\n",
    "        log.close()\n",
    "\n",
    "        # RETRIEVE DBNSFP PART WITH ALL ANNOTATIONS\n",
    "        dbnsfp_annotation_dask = pd.read_csv('/gstock/biolo_datasets/variation/benchmark/Annot_datasets/dbNSFP/v4.2/dbNSFP4.2a_variant.chr{}.gz'.format(chromosome), compression='gzip', sep='\\t', low_memory=False)\n",
    "        dbnsfp_annotation_dask['FULL_ID'] = dbnsfp_annotation_dask['#chr'].astype(str) + '-' + dbnsfp_annotation_dask['pos(1-based)'].astype(str)  + '-' + dbnsfp_annotation_dask['ref'].astype(str) + '-' + dbnsfp_annotation_dask['alt'].astype(str) + '-' + dbnsfp_annotation_dask['hg19_chr'].astype(str)  + '-' + dbnsfp_annotation_dask['hg19_pos(1-based)'].astype(str)\n",
    "\n",
    "\n",
    "        # RETRIEVE FEATURE COLS\n",
    "        base_cols = ['FULL_ID', '#chr', 'pos(1-based)', 'ref', 'alt', 'aaref', 'aaalt', 'rs_dbSNP', 'hg19_chr', 'hg19_pos(1-based)', 'hg18_chr', 'hg18_pos(1-based)', 'aapos', 'genename', 'Ensembl_geneid', 'Ensembl_transcriptid', 'Ensembl_proteinid']\n",
    "        feature_cols = sorted([\"gnomAD_exomes_AF\", \"MetaSVM_score\", \"MetaLR_score\", \"VEST4_score\", \"gnomAD_exomes_ASJ_AF\", \"gnomAD_exomes_FIN_AF\", \"gnomAD_exomes_NFE_AF\", \"MPC_score\", \"CADD_phred\", \"SIFT_score\", \"phyloP100way_vertebrate\", \"29way_logOdds\", \"Polyphen2_HVAR_score\", \"GERP++_RS\", \"phastCons17way_primate\", \"phastCons30way_mammalian\", \"phyloP30way_mammalian\", \"gnomAD_exomes_SAS_AF\", \"phastCons100way_vertebrate\", \"phyloP17way_primate\", \"gnomAD_exomes_AMR_AF\", \"gnomAD_exomes_EAS_AF\", \"gnomAD_exomes_AFR_AF\", ])\n",
    "        gnomad_cols = [e for e in feature_cols if 'gnomAD' in e ]\n",
    "\n",
    "        dbnsfp_annotation_dask = dbnsfp_annotation_dask.rename({'SiPhy_29way_logOdds' : '29way_logOdds'}, axis=1)\n",
    "        dbnsfp_annotation_lite = dbnsfp_annotation_dask[base_cols + feature_cols]\n",
    "        dbnsfp_annotation_lite = dbnsfp_annotation_lite.replace('.', np.nan)\n",
    "        dbnsfp_annotation_lite[gnomad_cols] = dbnsfp_annotation_lite[gnomad_cols].fillna(0)\n",
    "        dbnsfp_annotation_lite = dbnsfp_annotation_lite[base_cols + feature_cols]\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), '- dbNSFP lite  ', dbnsfp_annotation_lite.shape))\n",
    "        log.close()\n",
    "\n",
    "\n",
    "        # MERGE ON VARIANT ID\n",
    "        dbnsfp_annotation_complete_dask = pd.merge(dbnsfp_vcf_hail_pandas_tmp, dbnsfp_annotation_lite,on=['FULL_ID'], how='left')\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), '- VEP VCF X dbNSFP lite - ', dbnsfp_annotation_complete_dask.shape))\n",
    "        log.close()\n",
    "\n",
    "        base_cols = base_cols + ['MISTIC', 'IMPACT', 'Consequence']\n",
    "#         print(dbnsfp_annotation_complete_dask.columns.tolist())\n",
    "#         print(dbnsfp_annotation_complete_dask[['FULL_ID', 'IMPACT', 'Consequence', 'MISTIC']])\n",
    "        \n",
    "        # RETRIEVE VARIANTS WITHOUT MISTIC SCORE\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic = dbnsfp_annotation_complete_dask.loc[(dbnsfp_annotation_complete_dask['MISTIC'].isna() == True)]\n",
    "        dbnsfp_annotation_complete_dask_others = dbnsfp_annotation_complete_dask.loc[~dbnsfp_annotation_complete_dask.index.isin(dbnsfp_annotation_complete_dask_missense_mistic.index.tolist())]\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), ' - missense WITHOUT MISTIC - ',dbnsfp_annotation_complete_dask_missense_mistic.shape))\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), ' - OTHERS - ',dbnsfp_annotation_complete_dask_others.shape))\n",
    "        log.close()\n",
    "\n",
    "\n",
    "        # HANDLE CONDEL SPECIFIC CASE\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic['CONDEL'] = dbnsfp_annotation_complete_dask_missense_mistic['CONDEL'].fillna(\"['neutral(0.5)']\").apply(lambda r: np.mean([float(e.split('(')[1].split(')')[0]) for e in eval(r) if e != \"\"]))\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic = dbnsfp_annotation_complete_dask_missense_mistic.rename({'CONDEL' : 'Condel'}, axis=1)\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), '- Selecting values ... - '))\n",
    "        log.close()\n",
    "        \n",
    "        # COMPUTE MEDIAN FOR ROWS WITH MULTIPLE VALUES\n",
    "        def select_values(r):\n",
    "             return r.apply(lambda e: np.median([float(sub_e) for sub_e in str(e).split(';') if sub_e != '.']))\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic[feature_cols] = dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].apply(lambda r: select_values(r), axis=1)\n",
    "\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), '- Selecting values finished ! - '))\n",
    "        log.close()\n",
    "        \n",
    "        feature_cols = sorted(feature_cols + ['CCR', 'Condel'])\n",
    "\n",
    "        # MEDIAN DICT IF ANNOTATION NOT AVAILABLE\n",
    "        ## WAS PRECOMPUTED ON dbNSFP DURING MISTIC DVT\n",
    "        median_dict = {\n",
    "                \"29way_logOdds\": 14,\n",
    "                \"CADD_phred\": 20,\n",
    "                \"CONDEL\": 0.5,\n",
    "                \"CCR\" : 0,\n",
    "                \"GERP++_RS\": 4.8,\n",
    "                \"Grantham\": 70,\n",
    "                \"HI_score\": 0.25,\n",
    "                \"MetaLR_score\": 0.5,\n",
    "                \"MetaSVM_score\": 0.82257,\n",
    "                \"MPC_score\" : 0.5264,\n",
    "                \"phastCons100way_vertebrate\": 1,\n",
    "                \"phastCons17way_primate\": 0.8,\n",
    "                \"phastCons30way_mammalian\": 0.9,\n",
    "                \"phyloP100way_vertebrate\": 5.5,\n",
    "                \"phyloP17way_primate\": 0.6,\n",
    "                \"phyloP30way_mammalian\": 1.1,\n",
    "                \"Polyphen2_HVAR_score\": 0.8,\n",
    "                \"SIFT_score\": 0.05,\n",
    "                \"VEST4_score\": 0.5,\n",
    "        }\n",
    "        # FILLNA WITH MEDIANS\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic[feature_cols] = dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].fillna(median_dict)\n",
    "\n",
    "\n",
    "        # DROP DUPLICATES ON AA\n",
    "        combi_aa_missense = dbnsfp_annotation_dask[['aaref', 'aaalt']].drop_duplicates()\n",
    "        # RETRIEVE AAINDEX SCORES\n",
    "        combi_aa_missense = combi_aa_missense.reset_index(drop=True).apply(lambda r : aa_index_pandas(r), axis=1)\n",
    "        aaindex_to_keep = [\"HUTJ700101\", \"KOSJ950114\", \"LINK010101\", \"OVEJ920104\", \"KOSJ950109\", \"KOSJ950112\", \"ZHAC000102\", \"OVEJ920105\", \"KOSJ950110\", \"KOSJ950113\", \"KOSJ950106\", \"KOSJ950105\", \"ZHAC000105\", \"KOSJ950115\", \"KOSJ950101\", \"KOSJ950108\", \"KOSJ950111\", \"KOSJ950104\", \"DOSZ010102\", \"QU_C930101\", \"TOBD000102\", \"MOOG990101\", \"KESO980102\", \"BRYS930101\", \"SIMK990104\", \"NIEK910102\", \"MIYT790101\", \"NIEK910101\", \"VENM980101\", \"JOHM930101\", \"MIYS930101\", \"SIMK990103\", \"BENS940104\", \"QU_C930102\", \"THOP960101\", \"LUTR910106\", \"OGAK980101\", \"KANM000101\", \"BONM030104\", \"QUIB020101\", \"TANS760101\", \"RISJ880101\", \"MIYS960101\", \"ZHAC000106\", \"MIRL960101\", \"KAPO950101\", \"PRLA000102\", \"CSEM940101\", \"MIYS850102\", \"BENS940102\", \"MIYS850103\", \"SKOJ000101\", \"MIYS960102\", \"LUTR910101\", \"MIYS990107\", \"LUTR910107\", \"LUTR910104\", \"MUET010101\", \"LUTR910102\", \"SKOJ970101\", \"RUSR970101\", \"BONM030102\", \"NGPC000101\", \"BENS940103\", \"AZAE970101\", \"VOGG950101\", \"AZAE970102\", \"DAYM780302\", \"JOND940101\", \"GONG920101\", \"GEOD900101\", \"HENS920102\", \"RUSR970102\", \"MOHR870101\", \"NAOD960101\", \"QU_C930103\", \"MUET020101\", \"ALTS910101\", \"DAYM780301\", \"OVEJ920101\", \"JOND920103\", \"WEIL970102\", \"HENS920104\", \"WEIL970101\", \"CROG050101\", \"FEND850101\", \"HENS920101\", \"MUET020102\", \"MCLA720101\", \"FITW660101\", ]\n",
    "        combi_aa_missense = combi_aa_missense[['aaref', 'aaalt'] + aaindex_to_keep]\n",
    "\n",
    "        # MERGE DATAFRAMES\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic = pd.merge(dbnsfp_annotation_complete_dask_missense_mistic, combi_aa_missense, on=['aaref', 'aaalt'])\n",
    "\n",
    "        feature_cols = sorted(feature_cols + aaindex_to_keep)\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic[feature_cols] = dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].fillna(dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].median())\n",
    "\n",
    "\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic = dbnsfp_annotation_complete_dask_missense_mistic[base_cols + feature_cols]\n",
    "\n",
    "        \n",
    "        # PREDICT MISTIC MISSING SCORES\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic['MISTIC_VC'] = mistic_vc.predict_proba(dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].values)[:,1]\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic['MISTIC_LR'] = mistic_lr.predict_proba(dbnsfp_annotation_complete_dask_missense_mistic[feature_cols].values)[:,1]\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic.loc[dbnsfp_annotation_complete_dask_missense_mistic['gnomAD_exomes_AF'] == 0, 'MISTIC'] = dbnsfp_annotation_complete_dask_missense_mistic.loc[dbnsfp_annotation_complete_dask_missense_mistic['gnomAD_exomes_AF'] == 0]['MISTIC_LR']\n",
    "        dbnsfp_annotation_complete_dask_missense_mistic.loc[dbnsfp_annotation_complete_dask_missense_mistic['gnomAD_exomes_AF'] != 0, 'MISTIC'] = dbnsfp_annotation_complete_dask_missense_mistic.loc[dbnsfp_annotation_complete_dask_missense_mistic['gnomAD_exomes_AF'] != 0]['MISTIC_VC']\n",
    "\n",
    "        # CONCAT MISTIC & OTHERS\n",
    "        dbnsfp_annotation_complete_chr_final = pd.concat([dbnsfp_annotation_complete_dask_missense_mistic, dbnsfp_annotation_complete_dask_others])[['FULL_ID', 'MISTIC', 'IMPACT', 'Consequence', 'aaref', 'aaalt']]\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), \" - CONCAT FINAL : \", dbnsfp_annotation_complete_chr_final.shape))\n",
    "        log.close()\n",
    "        \n",
    "        # FINAL FILE FOR OUTPUT\n",
    "        dbnsfp_annotation_complete_chr_final['#chr'] = dbnsfp_annotation_complete_chr_final['FULL_ID'].apply(lambda r: r.split('-')[0].replace('chr',''))\n",
    "        dbnsfp_annotation_complete_chr_final['#chr'] = dbnsfp_annotation_complete_chr_final['#chr'].astype(str)\n",
    "        dbnsfp_annotation_complete_chr_final['pos(1-based)'] = dbnsfp_annotation_complete_chr_final['FULL_ID'].apply(lambda r: int(r.split('-')[1]))\n",
    "        dbnsfp_annotation_complete_chr_final['ref'] = dbnsfp_annotation_complete_chr_final['FULL_ID'].apply(lambda r: r.split('-')[2])\n",
    "        dbnsfp_annotation_complete_chr_final['alt'] = dbnsfp_annotation_complete_chr_final['FULL_ID'].apply(lambda r: r.split('-')[3])\n",
    "        dbnsfp_annotation_complete_chr_final = dbnsfp_annotation_complete_chr_final.rename({'MISTIC' : 'score'}, axis=1)\n",
    "        dbnsfp_annotation_complete_chr_final = dbnsfp_annotation_complete_chr_final.dropna(subset=['aaref', 'aaalt'])\n",
    "        dbnsfp_annotation_complete_chr_final['comments'] = dbnsfp_annotation_complete_chr_final['aaref'] + '/' + dbnsfp_annotation_complete_chr_final['aaalt']\n",
    "        dbnsfp_annotation_complete_chr_final\n",
    "        dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['comments'].str.contains('X'), 'comments'] = dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['comments'].str.contains('X'), 'comments'] + \" - STOP GAINED/LOST / Hypothetic score\"\n",
    "        dbnsfp_annotation_complete_chr_final_bak = dbnsfp_annotation_complete_chr_final.copy()\n",
    "#         print(dbnsfp_annotation_complete_chr_final_bak)\n",
    "        \n",
    "#         dbnsfp_annotation_complete_chr_final.loc[(dbnsfp_annotation_complete_chr_final['IMPACT'].str.contains('HIGH')) & (dbnsfp_annotation_complete_chr_final['score'].isna() == True), 'Suppl_comments'] = 'No missense in VEP predictions / HIGH impact'\n",
    "#         dbnsfp_annotation_complete_chr_final.loc[(dbnsfp_annotation_complete_chr_final['IMPACT'].str.contains('LOW|MODIFIER')) & (dbnsfp_annotation_complete_chr_final['score'].isna() == True) & (dbnsfp_annotation_complete_chr_final['Suppl_comments'].isna() == True), 'Suppl_comments'] = 'No missense in VEP predictions / LOW|MODIFIER impact'\n",
    "#         dbnsfp_annotation_complete_chr_final['Suppl_comments'] = dbnsfp_annotation_complete_chr_final['Suppl_comments'].fillna('')\n",
    "#         dbnsfp_annotation_complete_chr_final['comments'] = dbnsfp_annotation_complete_chr_final['comments'].astype(str) + ' - ' + dbnsfp_annotation_complete_chr_final['Suppl_comments'].astype(str)\n",
    "        dbnsfp_annotation_complete_chr_final_bak2 = dbnsfp_annotation_complete_chr_final.copy()\n",
    "#         print(dbnsfp_annotation_complete_chr_final_bak2)\n",
    "        \n",
    "#         print(dbnsfp_annotation_complete_chr_final.columns)\n",
    "\n",
    "\n",
    "        # COMPUTE TEXT PREDICTIONS\n",
    "        dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['score'] >= 0.5, 'pred'] = 'D'\n",
    "        dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['score'] < 0.5, 'pred'] = 'T'\n",
    "        dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['score'].isna() == True, 'pred'] = 'U'\n",
    "\n",
    "\n",
    "#         dbnsfp_annotation_complete_chr_final.loc[(dbnsfp_annotation_complete_chr_final['IMPACT'].str.contains('HIGH')) & (dbnsfp_annotation_complete_chr_final['score'].isna() == True) , 'score'] = 1\n",
    "#         dbnsfp_annotation_complete_chr_final.loc[(dbnsfp_annotation_complete_chr_final['IMPACT'].str.contains('LOW|MODIFIER')) & (dbnsfp_annotation_complete_chr_final['score'].isna() == True) , 'score'] = 0\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), \" - FINAL CLEAN : \", dbnsfp_annotation_complete_chr_final.shape))\n",
    "        log.close()\n",
    "\n",
    "        dbnsfp_annotation_complete_chr_final['sd'] = 0\n",
    "        dbnsfp_annotation_complete_chr_final['comments'] = dbnsfp_annotation_complete_chr_final['comments'].fillna('*')\n",
    "        \n",
    "        \n",
    "        template = pd.read_csv(\"/gstock/MISTIC/CAGI6/download/templates/dbNSFP4_nsSNV.chr{}.template.gz\".format(chromosome), compression='gzip', sep='\\t')\n",
    "        template['ID'] = template['#chr'].astype(str) + \"-\" + template['pos(1-based)'].astype(str) + \"-\" + template['ref'].astype(str) + \"-\" + template['alt'].astype(str)\n",
    "\n",
    "        dbnsfp_annotation_complete_chr_final = dbnsfp_annotation_complete_chr_final.dropna(subset=['aaref', 'aaalt'])\n",
    "        dbnsfp_annotation_complete_chr_final['ID'] = dbnsfp_annotation_complete_chr_final['#chr'].astype(str) + \"-\" + dbnsfp_annotation_complete_chr_final['pos(1-based)'].astype(str) + \"-\" + dbnsfp_annotation_complete_chr_final['ref'].astype(str) + \"-\" + dbnsfp_annotation_complete_chr_final['alt'].astype(str)\n",
    "        dbnsfp_annotation_complete_chr_final = dbnsfp_annotation_complete_chr_final.loc[dbnsfp_annotation_complete_chr_final['ID'].isin(template['ID'].values.tolist())]\n",
    "        dbnsfp_annotation_complete_chr_final = dbnsfp_annotation_complete_chr_final[['#chr', 'pos(1-based)','ref','alt','score','sd','pred', 'comments']].drop_duplicates().sort_values(by=['#chr', 'pos(1-based)', 'ref', 'alt', 'comments'])\n",
    "        \n",
    "        dbnsfp_annotation_complete_chr_final[['#chr', 'pos(1-based)','ref','alt','score','sd','pred', 'comments']].to_csv(output_file_path, sep='\\t', index=False)\n",
    "        log = open('/gstock/MISTIC/CAGI6/download/OUTPUT/log.txt', 'a')\n",
    "        log.write(\"{}{}\\n\".format(time.strftime(\"%m/%d/%Y, %H:%M:%S\", time.localtime()), ' - FINAL EXPORTED'))\n",
    "        log.close()\n",
    "        \n",
    "    #     test[['#chr', 'pos(1-based)','ref','alt','score','sd','pred', 'comments', 'IMPACT', 'Consequence']]\n",
    "\n",
    "#         subprocess_args = [\n",
    "#             \"python\",\n",
    "#             \"/gstock/MISTIC/CAGI6/download/validation.py\",\n",
    "#             output_file_path,\n",
    "#         ]\n",
    "#         p1 = subprocess.Popen(subprocess_args, stdout=subprocess.PIPE,)\n",
    "#         p1.wait()\n",
    "    else:\n",
    "        print('File exists for chr {}'.format(chromosome))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "dcf2ea13-34c1-4a79-8d7e-330eda927f00",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Welcome, weber8thomas!\n",
      "\n",
      "MISTIC_chr1.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr10.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr11.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr12.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr13.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr14.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr15.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr16.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr17.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr18.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr19.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr2.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr20.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr21.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr22.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr3.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr4.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr5.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr6.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr7.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr8.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chr9.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chrM.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chrX.tsv\n",
      "\n",
      "##################################################\n",
      " Uploading file to Synapse storage \n",
      "##################################################\n",
      "\n",
      "MISTIC_chrY.tsv\n"
     ]
    }
   ],
   "source": [
    "# EXPORT TO SYNAPSE\n",
    "\n",
    "import synapseclient\n",
    "from synapseclient import File\n",
    "syn = synapseclient.login(\"weber8thomas\", \"alpiance\")\n",
    "\n",
    "# Add a local file to an existing project (syn12345) on Synapse\n",
    "\n",
    "dir_path = \"/gstock/MISTIC/CAGI6/download/OUTPUT/\"\n",
    "list_dir = sorted([e for e in os.listdir(dir_path) if e.endswith('.tsv')])\n",
    "for file in list_dir:\n",
    "    print(file)\n",
    "    file = File(path= dir_path + file, parent='syn26264707')\n",
    "\n",
    "    file = syn.store(file)\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
